diff -Naur orig/mate-tweak patched/mate-tweak
--- orig/mate-tweak	2015-07-22 01:19:04.000000000 +0200
+++ patched/mate-tweak	2015-07-22 18:32:40.126303002 +0200
@@ -481,7 +481,8 @@
             mint_menu_available = True
 
         gnome_menu_available = False
-        if os.path.exists('/usr/lib/gnome-main-menu/main-menu'):
+        if os.path.exists('/usr/libexec/mate/main-menu') or \
+           os.path.exists('/usr/lib/gnome-main-menu/main-menu'):
             gnome_menu_available = True
 
         dock = False
