diff -Naur orig/data/xfdashboard-autostart.desktop.in.in patched/data/xfdashboard-autostart.desktop.in.in
--- orig/data/xfdashboard-autostart.desktop.in.in	2015-04-09 06:31:58.000000000 +0200
+++ patched/data/xfdashboard-autostart.desktop.in.in	2015-04-13 22:29:26.459501314 +0200
@@ -5,6 +5,6 @@
 TryExec=@bindir@/xfdashboard
 Exec=@bindir@/xfdashboard -d
 Icon=xfdashboard
-NotShowIn=GNOME;KDE;Unity;
+OnlyShowIn=XFCE;
 Terminal=false
 Hidden=true
diff -Naur orig/data/xfdashboard.desktop.in.in patched/data/xfdashboard.desktop.in.in
--- orig/data/xfdashboard.desktop.in.in	2015-02-02 20:58:05.000000000 +0100
+++ patched/data/xfdashboard.desktop.in.in	2015-04-13 22:29:32.963533566 +0200
@@ -7,5 +7,5 @@
 Exec=@bindir@/xfdashboard
 Icon=xfdashboard
 Categories=System;
-NotShowIn=GNOME;KDE;Unity;
+OnlyShowIn=XFCE;
 Terminal=false
