diff -Naur orig/uhu-installer mod/uhu-installer
--- orig/uhu-installer	2014-06-19 13:10:49.000000000 +0200
+++ mod/uhu-installer	2015-09-16 21:54:13.530773976 +0200
@@ -1028,9 +1028,17 @@
 			name => _('Mounting disks'),
 			func => sub {
 				for (@$umounts) {
+					if ($_->[1] eq '/mnt/') {
+						_system 'mkdir', '-p', $_->[1];
+						_system 'mount', $_->[0], $_->[1];
+					}
+				}
+				for (@$umounts) {
+				if ($_->[1] ne '/mnt/') {
 					_system 'mkdir', '-p', $_->[1];
 					_system 'mount', $_->[0], $_->[1];
 				}
+				}
 			}
 		};
 		
