diff -Nur orig/installer/ep128emu mod/installer/ep128emu
--- orig/installer/ep128emu	2016-10-19 20:03:30.000000000 +0200
+++ mod/installer/ep128emu	2016-10-19 22:50:48.851806071 +0200
@@ -7,7 +7,7 @@
       cp /usr/share/ep128emu/roms/ep128emu_roms-2.0.10.bin ~/.ep128emu/roms/
     else
       curl -o "$HOME/.ep128emu/roms/ep128emu_roms-2.0.10.bin"   \
-      "http://ep128emu.enterpriseforever.com/roms/ep128emu_roms-2.0.10.bin"
+      "https://enterpriseforever.com/letoltesek-downloads/egyeb-misc/?action=dlattach;attach=16433"
     fi
   epmakecfg "$HOME/.ep128emu"
   fi
diff -Nur orig/resource/makecfg-wrapper mod/resource/makecfg-wrapper
--- orig/resource/makecfg-wrapper	2016-10-19 20:03:30.000000000 +0200
+++ mod/resource/makecfg-wrapper	2016-10-19 22:28:31.170334373 +0200
@@ -10,6 +10,6 @@
 rom="$HOME/.ep128emu/roms/ep128emu_roms-2.0.10.bin"
 if [ ! -s "$rom" ]; then
 	echo "Downloading ROM to $rom"
-	wget -O "$rom" http://ep128emu.enterpriseforever.com/roms/ep128emu_roms-2.0.10.bin || exit 1
+	wget -O "$rom" https://enterpriseforever.com/letoltesek-downloads/egyeb-misc/?action=dlattach\;attach=16433 || exit 1
 fi
 exec $0.bin
