diff -Naur orig/lxqt-config-session/CMakeLists.txt mod/lxqt-config-session/CMakeLists.txt
--- orig/lxqt-config-session/CMakeLists.txt	2014-07-21 15:45:00.000000000 +0200
+++ mod/lxqt-config-session/CMakeLists.txt	2014-10-01 19:35:26.688080983 +0200
@@ -3,7 +3,7 @@
 add_definitions ( -Wall ) # FIXME: this is compiler specific
 
 if(USE_QT5)
-	set(QTX_INCLUDE_DIRS "")
+	set(QTX_INCLUDE_DIRS "/usr/include/qt5xdg")
 	set(QTX_LIBRARIES Qt5::Widgets Qt5::DBus Qt5::X11Extras)
 else()
 	set(QTX_INCLUDE_DIRS ${QT_QTCORE_INCLUDE_DIR} ${QT_QTGUI_INCLUDE_DIR} ${QT_QTDBUS_INCLUDE_DIR} ${QT_QTXML_INCLUDE_DIR})
diff -Naur orig/lxqt-session/CMakeLists.txt mod/lxqt-session/CMakeLists.txt
--- orig/lxqt-session/CMakeLists.txt	2014-07-21 15:45:00.000000000 +0200
+++ mod/lxqt-session/CMakeLists.txt	2014-10-01 19:37:25.832076249 +0200
@@ -7,7 +7,7 @@
 add_definitions ( -Wall ) # FIXME: this is compiler specific
 
 if(USE_QT5)
-	set(QTX_INCLUDE_DIRS "")
+	set(QTX_INCLUDE_DIRS "/usr/include/qt5xdg")
 	set(QTX_LIBRARIES Qt5::Widgets Qt5::DBus Qt5::X11Extras)
 else()
 	set(QTX_INCLUDE_DIRS ${QT_QTCORE_INCLUDE_DIR} ${QT_QTGUI_INCLUDE_DIR} ${QT_QTDBUS_INCLUDE_DIR} ${QT_QTXML_INCLUDE_DIR})
