diff -Naur orig/external/libcdr/ExternalProject_libcdr.mk patched/external/libcdr/ExternalProject_libcdr.mk
--- orig/external/libcdr/ExternalProject_libcdr.mk	2015-10-24 17:21:49.000000000 +0200
+++ patched/external/libcdr/ExternalProject_libcdr.mk	2015-12-09 11:55:58.769288271 +0100
@@ -36,7 +36,8 @@
 			--disable-werror \
 			--disable-weffc \
 			$(if $(VERBOSE)$(verbose),--disable-silent-rules,--enable-silent-rules) \
-			CXXFLAGS="$(if $(SYSTEM_BOOST),$(BOOST_CPPFLAGS),-I$(call gb_UnpackedTarball_get_dir,boost))" \
+			CXXFLAGS="$(if $(SYSTEM_BOOST),$(BOOST_CPPFLAGS),-I$(call gb_UnpackedTarball_get_dir,boost)) \
+				-DBOOST_ERROR_CODE_HEADER_ONLY -DBOOST_SYSTEM_NO_DEPRECATED" \
 			$(if $(CROSS_COMPILING),--build=$(BUILD_PLATFORM) --host=$(HOST_PLATFORM)) \
 		&& $(MAKE) \
 	)
