diff -Naur orig/example.jwmrc mod/example.jwmrc
--- orig/example.jwmrc	2014-03-15 20:00:14.000000000 +0100
+++ mod/example.jwmrc	2015-03-27 14:04:20.000000000 +0100
@@ -4,12 +4,12 @@
 
     <!-- The root menu. -->
     <RootMenu onroot="12">
-        <Program icon="terminal.png" label="Terminal">xterm</Program>
-        <Menu icon="folder.png" label="Applications">
+        <Program icon="xterm_48x48.xpm" label="xterm">xterm</Program>
+        <Menu icon="folder.png" label="Alkalmazások">
             <Program icon="music.png" label="Audacious">audacious</Program>
             <Program icon="calculator.png" label="Calculator">xcalc</Program>
             <Program icon="gimp.png" label="Gimp">gimp</Program>
-            <Program icon="www.png" label="Iceweasel">iceweasel</Program>
+            <Program icon="www.png" label="Firefox">firefox</Program>
             <Program icon="editor.png" label="XEdit">xedit</Program>
         </Menu>
         <Menu icon="folder.png" label="Utilities">
@@ -120,7 +120,8 @@
          for icons in multiple paths.
       -->
     <IconPath>
-        /usr/share/icons/wm-icons/32x32-gant
+        /usr/share/icons/hicolor/48x48/
+	/usr/share/pixmaps
     </IconPath>
 
     <!-- Virtual Desktops -->
@@ -130,7 +131,7 @@
               contained within a Desktop tag to give a specific background
               for that desktop.
          -->
-        <Background type="image">$HOME/images/bg.jpg</Background>
+        <Background type="image">/usr/share/wallpapers/UHU/blur.jpg</Background>
     </Desktops>
 
     <!-- Double click speed (in milliseconds) -->
