diff -Naur orig/libfm-qt/translations/libfm-qt_hu.ts mod/libfm-qt/translations/libfm-qt_hu.ts
--- orig/libfm-qt/translations/libfm-qt_hu.ts	2015-11-02 12:34:16.000000000 +0100
+++ mod/libfm-qt/translations/libfm-qt_hu.ts	2015-11-07 22:55:46.000000000 +0100
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.1" language="hu_HU">
+<TS version="2.1" language="hu">
 <context>
     <name>AppChooserDialog</name>
     <message>
@@ -310,12 +310,12 @@
     <message>
         <location filename="../createnewmenu.cpp" line="29"/>
         <source>Folder</source>
-        <translation type="unfinished">Mappa</translation>
+        <translation>Mappa</translation>
     </message>
     <message>
         <location filename="../createnewmenu.cpp" line="33"/>
         <source>Blank File</source>
-        <translation type="unfinished">Üres fájl</translation>
+        <translation>Üres fájl</translation>
     </message>
 </context>
 <context>
@@ -336,17 +336,17 @@
     <message>
         <location filename="../dirtreeview.cpp" line="208"/>
         <source>Open in New T&amp;ab</source>
-        <translation type="unfinished"></translation>
+        <translation>Megnyi&amp;tás új fülön</translation>
     </message>
     <message>
         <location filename="../dirtreeview.cpp" line="212"/>
         <source>Open in New Win&amp;dow</source>
-        <translation type="unfinished"></translation>
+        <translation>Me&amp;gnyitás új ablakban</translation>
     </message>
     <message>
         <location filename="../dirtreeview.cpp" line="217"/>
         <source>Open in Termina&amp;l</source>
-        <translation type="unfinished"></translation>
+        <translation>Megnyitás terminá&amp;lban</translation>
     </message>
 </context>
 <context>
@@ -405,7 +405,7 @@
     <message>
         <location filename="../filemenu.cpp" line="132"/>
         <source>Create &amp;New</source>
-        <translation type="unfinished">&amp;Új létrehozása</translation>
+        <translation>&amp;Új létrehozása</translation>
     </message>
     <message>
         <location filename="../filemenu.cpp" line="151"/>
@@ -641,17 +641,17 @@
     <message>
         <location filename="../filesearchdialog.cpp" line="120"/>
         <source>Error</source>
-        <translation type="unfinished">Hiba</translation>
+        <translation>Hiba</translation>
     </message>
     <message>
         <location filename="../filesearchdialog.cpp" line="120"/>
         <source>You should add at least add one directory to search.</source>
-        <translation type="unfinished"></translation>
+        <translation>Legalább egy mappa megadása kell a kereséshez.</translation>
     </message>
     <message>
         <location filename="../filesearchdialog.cpp" line="127"/>
         <source>Select a folder</source>
-        <translation type="unfinished"></translation>
+        <translation>Mappa választás</translation>
     </message>
 </context>
 <context>
@@ -856,32 +856,32 @@
     <message>
         <location filename="../placesview.cpp" line="346"/>
         <source>Open in New Tab</source>
-        <translation type="unfinished"></translation>
+        <translation>Megnyitás új fülön</translation>
     </message>
     <message>
         <location filename="../placesview.cpp" line="349"/>
         <source>Open in New Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Megnyitás új ablakban</translation>
     </message>
     <message>
         <location filename="../placesview.cpp" line="367"/>
         <source>Move Bookmark Up</source>
-        <translation type="unfinished"></translation>
+        <translation>Könyvjelző föl</translation>
     </message>
     <message>
         <location filename="../placesview.cpp" line="372"/>
         <source>Move Bookmark Down</source>
-        <translation type="unfinished"></translation>
+        <translation>Könyvjelző le</translation>
     </message>
     <message>
         <location filename="../placesview.cpp" line="376"/>
         <source>Rename Bookmark</source>
-        <translation type="unfinished"></translation>
+        <translation>Könyvjelző átnevezés</translation>
     </message>
     <message>
         <location filename="../placesview.cpp" line="379"/>
         <source>Remove Bookmark</source>
-        <translation type="unfinished"></translation>
+        <translation>Könyvjelző törlés</translation>
     </message>
     <message>
         <location filename="../placesview.cpp" line="388"/>
@@ -1033,7 +1033,7 @@
     <message>
         <location filename="../utilities.cpp" line="164"/>
         <source>Create Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>Mappa létrehozás</translation>
     </message>
     <message>
         <location filename="../utilities.cpp" line="169"/>
@@ -1119,176 +1119,176 @@
     <message>
         <location filename="../filesearch.ui" line="14"/>
         <source>Search Files</source>
-        <translation type="unfinished"></translation>
+        <translation>Fájlkeresés</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="29"/>
         <source>Name/Location</source>
-        <translation type="unfinished"></translation>
+        <translation>Név/hely</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="35"/>
         <source>File Name Patterns:</source>
-        <translation type="unfinished"></translation>
+        <translation>Fájlnév szűrő:</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="41"/>
         <source>*</source>
-        <translation type="unfinished"></translation>
+        <translation>*</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="48"/>
         <source>Case insensitive</source>
-        <translation type="unfinished"></translation>
+        <translation>Nagybetű érzéketlen</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="55"/>
         <source>Use regular expression</source>
-        <translation type="unfinished"></translation>
+        <translation>Szabályos kifejezés</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="65"/>
         <source>Places to Search:</source>
-        <translation type="unfinished"></translation>
+        <translation>Keresési helyek:</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="78"/>
         <source>&amp;Add</source>
-        <translation type="unfinished"></translation>
+        <translation>Hozzá&amp;ad</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="90"/>
         <source>&amp;Remove</source>
-        <translation type="unfinished"></translation>
+        <translation>Tö&amp;röl</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="119"/>
         <source>Search in sub directories</source>
-        <translation type="unfinished"></translation>
+        <translation>Alkönyvtárakban keres</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="126"/>
         <source>Search for hidden files</source>
-        <translation type="unfinished"></translation>
+        <translation>Rejtett fájlok is</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="137"/>
         <source>File Type</source>
-        <translation type="unfinished"></translation>
+        <translation>Fájltípus</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="143"/>
         <source>Only search for files of following types:</source>
-        <translation type="unfinished"></translation>
+        <translation>Csak a következőkben keres:</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="149"/>
         <source>Text files</source>
-        <translation type="unfinished"></translation>
+        <translation>Szövegfájlok</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="156"/>
         <source>Image files</source>
-        <translation type="unfinished"></translation>
+        <translation>Képfájlok</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="163"/>
         <source>Audio files</source>
-        <translation type="unfinished"></translation>
+        <translation>Hangfájlok</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="170"/>
         <source>Video files</source>
-        <translation type="unfinished"></translation>
+        <translation>Videók</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="177"/>
         <source>Documents</source>
-        <translation type="unfinished"></translation>
+        <translation>Dokumentumok</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="184"/>
         <source>Folders</source>
-        <translation type="unfinished"></translation>
+        <translation>Könyvtárak</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="208"/>
         <source>Content</source>
-        <translation type="unfinished"></translation>
+        <translation>Tartalom</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="214"/>
         <source>File contains:</source>
-        <translation type="unfinished"></translation>
+        <translation>Fájl tartalmaz:</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="223"/>
         <source>Case insensiti&amp;ve</source>
-        <translation type="unfinished"></translation>
+        <translation>Nagybetű ér&amp;zéketlen</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="230"/>
         <source>&amp;Use regular expression</source>
-        <translation type="unfinished"></translation>
+        <translation>Szabvá&amp;nyos kifejezés</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="254"/>
         <source>Properties</source>
-        <translation type="unfinished">Tulajdonságok</translation>
+        <translation>Tulajdonságok</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="260"/>
         <source>File Size:</source>
-        <translation type="unfinished"></translation>
+        <translation>Fájlméret:</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="266"/>
         <source>Larger than:</source>
-        <translation type="unfinished"></translation>
+        <translation>Nagyobb mint:</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="282"/>
         <location filename="../filesearch.ui" line="323"/>
         <source>Bytes</source>
-        <translation type="unfinished"></translation>
+        <translation>Bájt</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="287"/>
         <location filename="../filesearch.ui" line="328"/>
         <source>KiB</source>
-        <translation type="unfinished"></translation>
+        <translation>Kb</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="292"/>
         <location filename="../filesearch.ui" line="333"/>
         <source>MiB</source>
-        <translation type="unfinished"></translation>
+        <translation>Mb</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="297"/>
         <location filename="../filesearch.ui" line="338"/>
         <source>GiB</source>
-        <translation type="unfinished"></translation>
+        <translation>Gb</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="307"/>
         <source>Smaller than:</source>
-        <translation type="unfinished"></translation>
+        <translation>Kissebb mint:</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="351"/>
         <source>Last Modified Time:</source>
-        <translation type="unfinished"></translation>
+        <translation>Utolsó módosítás ideje:</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="357"/>
         <source>Earlier than:</source>
-        <translation type="unfinished"></translation>
+        <translation>Korább mint:</translation>
     </message>
     <message>
         <location filename="../filesearch.ui" line="364"/>
         <source>Later than:</source>
-        <translation type="unfinished"></translation>
+        <translation>Később mint:</translation>
     </message>
 </context>
 </TS>
diff -Naur orig/pcmanfm/translations/pcmanfm-qt_hu.ts mod/pcmanfm/translations/pcmanfm-qt_hu.ts
--- orig/pcmanfm/translations/pcmanfm-qt_hu.ts	2015-11-02 12:34:16.000000000 +0100
+++ mod/pcmanfm/translations/pcmanfm-qt_hu.ts	2015-11-08 20:52:06.000000000 +0100
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.1" language="hu_HU">
+<TS version="2.1" language="hu">
 <context>
     <name>AboutDialog</name>
     <message>
@@ -11,6 +11,7 @@
     <message>
         <location filename="../../build/pcmanfm/ui_about.h" line="135"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;PCManFM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translatorcomment>Nincs értelme fordítani</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -19,6 +20,25 @@
         <translation>Könnyed fájlkezelő</translation>
     </message>
     <message>
+        <location filename="../../build/pcmanfm/ui_about.h" line="138"/>
+        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://lxqt.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;http://lxqt.org/&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translatorcomment>Nincs értelm fordítani</translatorcomment>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_about.h" line="139"/>
+        <source>Programming:
+* Hong Jen Yee (PCMan) &lt;pcman.tw@gmail.com&gt;
+</source>
+        <translation>Programfejlesztő:
+* Hong Jen Yee (PCMan) &lt;pcman.tw@gmail.com&gt;</translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_about.h" line="142"/>
+        <source>Authors</source>
+        <translation>Szerzők</translation>
+    </message>
+    <message>
         <location filename="../../build/pcmanfm/ui_about.h" line="143"/>
         <source>PCMan File Manager
 
@@ -37,27 +57,10 @@
 You should have received a copy of the GNU General Public License
 along with this program; if not, write to the Free Software
 Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.</source>
+        <translatorcomment>Ennek fordítása értelmetlen</translatorcomment>
         <translation></translation>
     </message>
     <message>
-        <location filename="../../build/pcmanfm/ui_about.h" line="139"/>
-        <source>Programming:
-* Hong Jen Yee (PCMan) &lt;pcman.tw@gmail.com&gt;
-</source>
-        <translation>Programfejlesztő:
-* Hong Jen Yee (PCMan) &lt;pcman.tw@gmail.com&gt;</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_about.h" line="138"/>
-        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://lxqt.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;http://lxqt.org/&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation></translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_about.h" line="142"/>
-        <source>Authors</source>
-        <translation>Szerzők</translation>
-    </message>
-    <message>
         <location filename="../../build/pcmanfm/ui_about.h" line="160"/>
         <source>License</source>
         <translation>Licensz</translation>
@@ -68,7 +71,7 @@
     <message>
         <location filename="../../build/pcmanfm/ui_autorun.h" line="110"/>
         <source>Removable medium is inserted</source>
-        <translation>Új hordozható adathordozó észlelve</translation>
+        <translation>Új cserélhető adathordozó észlelve</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_autorun.h" line="112"/>
@@ -96,32 +99,32 @@
     <message>
         <location filename="../../build/pcmanfm/ui_desktop-folder.h" line="75"/>
         <source>Form</source>
-        <translation type="unfinished"></translation>
+        <translation>Űrlap</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_desktop-folder.h" line="76"/>
         <source>Desktop</source>
-        <translation type="unfinished">Asztal</translation>
+        <translation>Asztal</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_desktop-folder.h" line="77"/>
         <source>Desktop folder:</source>
-        <translation type="unfinished">Asztal mappa:</translation>
+        <translation>Asztal mappa:</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_desktop-folder.h" line="79"/>
         <source>Image file</source>
-        <translation type="unfinished">Képfájl</translation>
+        <translation>Képfájl</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_desktop-folder.h" line="84"/>
         <source>Folder path</source>
-        <translation type="unfinished">Könyvtár útvonal</translation>
+        <translation>Könyvtár útvonal</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_desktop-folder.h" line="85"/>
         <source>&amp;Browse</source>
-        <translation type="unfinished">&amp;Böngészés</translation>
+        <translation>&amp;Böngészés</translation>
     </message>
 </context>
 <context>
@@ -202,18 +205,6 @@
         <translation>Asztalra kattintva az ablakkezelő menüje látszik</translation>
     </message>
     <message>
-        <source>Desktop</source>
-        <translation type="vanished">Asztal</translation>
-    </message>
-    <message>
-        <source>Desktop folder:</source>
-        <translation type="vanished">Asztal mappa:</translation>
-    </message>
-    <message>
-        <source>Folder path</source>
-        <translation type="vanished">Könyvtár útvonal</translation>
-    </message>
-    <message>
         <location filename="../../build/pcmanfm/ui_desktop-preferences.h" line="278"/>
         <source>Advanced</source>
         <translation>Haladó</translation>
@@ -227,6 +218,11 @@
         <translation>Fájlkezelő</translation>
     </message>
     <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="558"/>
+        <source>Go &amp;Up</source>
+        <translation>&amp;Föl</translation>
+    </message>
+    <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="560"/>
         <source>Go Up</source>
         <translation>Föl</translation>
@@ -237,35 +233,28 @@
         <translation>Alt+Föl</translation>
     </message>
     <message>
-        <source>Home</source>
-        <translation type="vanished">Saját könyvtár</translation>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="563"/>
+        <source>&amp;Home</source>
+        <translation>&amp;Saját könyvtár</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="564"/>
         <source>Alt+Home</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="583"/>
-        <source>Reload</source>
-        <translation>Újratölt</translation>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="565"/>
+        <source>&amp;Reload</source>
+        <translation>Új&amp;ratölt</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="566"/>
         <source>F5</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="563"/>
-        <source>&amp;Home</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="565"/>
-        <source>&amp;Reload</source>
-        <translation>Új&amp;ratölt</translation>
-    </message>
-    <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="567"/>
         <source>Go</source>
         <translation>Ugrás</translation>
@@ -281,6 +270,11 @@
         <translation>Ról&amp;am</translation>
     </message>
     <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="570"/>
+        <source>&amp;New Window</source>
+        <translation>Ú&amp;J ablak</translation>
+    </message>
+    <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="572"/>
         <source>New Window</source>
         <translation>ÚJ ablak</translation>
@@ -288,6 +282,7 @@
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="574"/>
         <source>Ctrl+N</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -298,6 +293,7 @@
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="576"/>
         <source>Ctrl+H</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -331,8 +327,39 @@
         <translation>&amp;Alkalmazások</translation>
     </message>
     <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="583"/>
+        <source>Reload</source>
+        <translation>Újratölt</translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="584"/>
+        <source>&amp;Icon View</source>
+        <translation>&amp;Ikonnézet</translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="585"/>
+        <source>&amp;Compact View</source>
+        <translation>&amp;Tömör nézet</translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="586"/>
+        <source>&amp;Detailed List</source>
+        <translation>&amp;Teljes lista</translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="587"/>
+        <source>&amp;Thumbnail View</source>
+        <translation>&amp;Bélyegképek</translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="588"/>
+        <source>Cu&amp;t</source>
+        <translation>&amp;Kivág</translation>
+    </message>
+    <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="589"/>
         <source>Ctrl+X</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -343,6 +370,7 @@
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="591"/>
         <source>Ctrl+C</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -353,6 +381,7 @@
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="593"/>
         <source>Ctrl+V</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -361,6 +390,12 @@
         <translation>&amp;Mindet</translation>
     </message>
     <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="595"/>
+        <source>Ctrl+A</source>
+        <translatorcomment>Jó</translatorcomment>
+        <translation></translation>
+    </message>
+    <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="596"/>
         <source>Pr&amp;eferences</source>
         <translation>B&amp;eállítások</translation>
@@ -368,135 +403,62 @@
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="597"/>
         <source>&amp;Ascending</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Növekvő</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="598"/>
         <source>&amp;Descending</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Csökkenő</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="599"/>
         <source>&amp;By File Name</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Fájlnév szerint</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="600"/>
         <source>By &amp;Modification Time</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Módosítási idő szerint</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="601"/>
         <source>By File &amp;Type</source>
-        <translation type="unfinished"></translation>
+        <translation>Fájl&amp;típus szerint</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="602"/>
         <source>By &amp;Owner</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Tulajdonos szerint</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="603"/>
         <source>&amp;Folder First</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="619"/>
-        <source>&amp;Invert Selection</source>
-        <translation>K&amp;ijelölés megfordítás</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="620"/>
-        <source>&amp;Delete</source>
-        <translation>&amp;Töröl</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="622"/>
-        <source>&amp;Rename</source>
-        <translation>&amp;Átnevez</translation>
+        <translation>Ma&amp;ppák elől</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="629"/>
         <source>&amp;Case Sensitive</source>
-        <translation type="unfinished"></translation>
+        <translation>Na&amp;gybetűérzékeny</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="630"/>
         <source>By File &amp;Size</source>
-        <translation type="unfinished"></translation>
+        <translation>Fá&amp;jlméret szerint</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="631"/>
         <source>&amp;Close Window</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="595"/>
-        <source>Ctrl+A</source>
-        <translation></translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="558"/>
-        <source>Go &amp;Up</source>
-        <translation>&amp;Föl</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="570"/>
-        <source>&amp;New Window</source>
-        <translation>&amp;ÚJ ablak</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="584"/>
-        <source>&amp;Icon View</source>
-        <translation>&amp;Ikonnézet</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="585"/>
-        <source>&amp;Compact View</source>
-        <translation>&amp;Tömör nézet</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="586"/>
-        <source>&amp;Detailed List</source>
-        <translation>&amp;Teljes lista</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="587"/>
-        <source>&amp;Thumbnail View</source>
-        <translation>&amp;Bélyegképek</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="588"/>
-        <source>Cu&amp;t</source>
-        <translation>&amp;Kivág</translation>
-    </message>
-    <message>
-        <source>Ascending</source>
-        <translation type="vanished">Növekvő</translation>
-    </message>
-    <message>
-        <source>Descending</source>
-        <translation type="vanished">Csökkenő</translation>
+        <translation>A&amp;blak bezárás</translation>
     </message>
     <message>
-        <source>By File Name</source>
-        <translation type="vanished">Név</translation>
-    </message>
-    <message>
-        <source>By Modification Time</source>
-        <translation type="vanished">Módosítási idő</translation>
-    </message>
-    <message>
-        <source>By File Type</source>
-        <translation type="vanished">Típus</translation>
-    </message>
-    <message>
-        <source>By Owner</source>
-        <translation type="vanished">Tulajdonos</translation>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="637"/>
+        <source>&amp;Folder</source>
+        <translation>&amp;Mappa</translation>
     </message>
     <message>
-        <source>Folder First</source>
-        <translation type="vanished">Mappák elől</translation>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="639"/>
+        <source>&amp;Blank File</source>
+        <translation>&amp;Üres fájl</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="604"/>
@@ -511,6 +473,7 @@
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="608"/>
         <source>Ctrl+T</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -544,13 +507,30 @@
         <translation>Alt+Jobb</translation>
     </message>
     <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="619"/>
+        <source>&amp;Invert Selection</source>
+        <translation>K&amp;ijelölés megfordítás</translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="620"/>
+        <source>&amp;Delete</source>
+        <translation>&amp;Töröl</translation>
+    </message>
+    <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="621"/>
         <source>Del</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="622"/>
+        <source>&amp;Rename</source>
+        <translation>&amp;Átnevez</translation>
+    </message>
+    <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="623"/>
         <source>F2</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -559,66 +539,26 @@
         <translation>Fü&amp;l bezárás</translation>
     </message>
     <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="626"/>
-        <source>File &amp;Properties</source>
-        <translation>&amp;Fájljellemzők</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="628"/>
-        <source>&amp;Folder Properties</source>
-        <translation>&amp;Mappajellemzők</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="638"/>
-        <source>Ctrl+Shift+N</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="640"/>
-        <source>Ctrl+Alt+N</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="643"/>
-        <source>Filter</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="646"/>
-        <source>C&amp;reate New</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="649"/>
-        <source>&amp;Sorting</source>
-        <translation>&amp;Rendezés</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="654"/>
-        <source>Main Toolbar</source>
-        <translation>Eszközsáv</translation>
-    </message>
-    <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="625"/>
         <source>Ctrl+W</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="626"/>
+        <source>File &amp;Properties</source>
+        <translation>&amp;Fájljellemzők</translation>
+    </message>
+    <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="627"/>
         <source>Alt+Return</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
-        <source>Case Sensitive</source>
-        <translation type="vanished">Nagybetűérzékeny</translation>
-    </message>
-    <message>
-        <source>By File Size</source>
-        <translation type="vanished">Fájlméret</translation>
-    </message>
-    <message>
-        <source>Close Window</source>
-        <translation type="vanished">Ablak bezárás</translation>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="628"/>
+        <source>&amp;Folder Properties</source>
+        <translation>&amp;Mappajellemzők</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="632"/>
@@ -633,6 +573,7 @@
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="634"/>
         <source>F4</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -646,29 +587,37 @@
         <translation>Könyvj&amp;elzőkezelés</translation>
     </message>
     <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="637"/>
-        <source>&amp;Folder</source>
-        <translation type="unfinished"></translation>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="638"/>
+        <source>Ctrl+Shift+N</source>
+        <translatorcomment>Jó</translatorcomment>
+        <translation></translation>
     </message>
     <message>
-        <location filename="../../build/pcmanfm/ui_main-win.h" line="639"/>
-        <source>&amp;Blank File</source>
-        <translation type="unfinished"></translation>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="640"/>
+        <source>Ctrl+Alt+N</source>
+        <translatorcomment>Jó</translatorcomment>
+        <translation></translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="641"/>
         <source>&amp;Find Files</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Fájlkeresés</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="642"/>
         <source>F3</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Jó</translatorcomment>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="643"/>
+        <source>Filter</source>
+        <translation>Szűrő</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="644"/>
         <source>Filter by string...</source>
-        <translation type="unfinished"></translation>
+        <translation>Szövegszűrő...</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="645"/>
@@ -676,6 +625,11 @@
         <translation>&amp;Fájl</translation>
     </message>
     <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="646"/>
+        <source>C&amp;reate New</source>
+        <translation>Ú&amp;j létrehozás</translation>
+    </message>
+    <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="647"/>
         <source>&amp;Help</source>
         <translation>&amp;Súgó</translation>
@@ -686,6 +640,11 @@
         <translation>&amp;Nézet</translation>
     </message>
     <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="649"/>
+        <source>&amp;Sorting</source>
+        <translation>Rendezé&amp;s</translation>
+    </message>
+    <message>
         <location filename="../../build/pcmanfm/ui_main-win.h" line="650"/>
         <source>&amp;Edit</source>
         <translation>Sz&amp;erkeszt</translation>
@@ -705,23 +664,30 @@
         <source>&amp;Tool</source>
         <translation>&amp;Eszközök</translation>
     </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_main-win.h" line="654"/>
+        <source>Main Toolbar</source>
+        <translation>Eszközsáv</translation>
+    </message>
 </context>
 <context>
     <name>PCManFM::Application</name>
     <message>
         <location filename="../application.cpp" line="162"/>
         <source>Name of configuration profile</source>
-        <translation>Beállítófájl neve</translation>
+        <translatorcomment>Beállítófájl neve</translatorcomment>
+        <translation></translation>
     </message>
     <message>
         <location filename="../application.cpp" line="162"/>
         <source>PROFILE</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
         <location filename="../application.cpp" line="165"/>
         <source>Run PCManFM as a daemon</source>
-        <translation>A PCManFM mint daemon fut</translation>
+        <translation>PCManFM daemon-ként</translation>
     </message>
     <message>
         <location filename="../application.cpp" line="168"/>
@@ -747,6 +713,7 @@
         <location filename="../application.cpp" line="177"/>
         <location filename="../application.cpp" line="193"/>
         <source>NAME</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -767,7 +734,8 @@
     <message>
         <location filename="../application.cpp" line="186"/>
         <source>FILE</source>
-        <translation>FILE</translation>
+        <translatorcomment>Jó</translatorcomment>
+        <translation></translation>
     </message>
     <message>
         <location filename="../application.cpp" line="190"/>
@@ -777,6 +745,7 @@
     <message>
         <location filename="../application.cpp" line="190"/>
         <source>MODE</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -792,6 +761,7 @@
     <message>
         <location filename="../application.cpp" line="196"/>
         <source>[FILE1, FILE2,...]</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -870,7 +840,7 @@
     <message>
         <location filename="../mainwindow.cpp" line="190"/>
         <source>Clear text (Ctrl+K)</source>
-        <translation type="unfinished"></translation>
+        <translation>Szövegtörlés (Ctrl+K)</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="455"/>
@@ -896,7 +866,7 @@
     <message>
         <location filename="../mainwindow.cpp" line="980"/>
         <source>Switch user command is not set.</source>
-        <translation>A felhasználót váltó alkalmazás beállítatlan.</translation>
+        <translation>Felhasználóváltó parancs határozatlan.</translation>
     </message>
 </context>
 <context>
@@ -1041,16 +1011,6 @@
         <translation>Ablak</translation>
     </message>
     <message>
-        <location filename="../../build/pcmanfm/ui_preferences.h" line="757"/>
-        <source>Default width of new windows:</source>
-        <translation>Új ablak alapszélessége:</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_preferences.h" line="758"/>
-        <source>Default height of new windows:</source>
-        <translation>Új ablak alapmagassága:</translation>
-    </message>
-    <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="754"/>
         <source>Always show the tab bar</source>
         <translation>Fülsáv mindig látható</translation>
@@ -1066,6 +1026,16 @@
         <translation>Utoljára bezárt ablak méretének megjegyzése</translation>
     </message>
     <message>
+        <location filename="../../build/pcmanfm/ui_preferences.h" line="757"/>
+        <source>Default width of new windows:</source>
+        <translation>Új ablak alapszélessége:</translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_preferences.h" line="758"/>
+        <source>Default height of new windows:</source>
+        <translation>Új ablak alapmagassága:</translation>
+    </message>
+    <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="724"/>
         <source>Browsing</source>
         <translation>Böngészés</translation>
@@ -1083,27 +1053,27 @@
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="727"/>
         <source>Default view mode:</source>
-        <translation>Alapértelmezett nézet:</translation>
+        <translation>Alapértelmezett nézetmód:</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="728"/>
         <source> sec</source>
-        <translation></translation>
+        <translation> mp</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="736"/>
         <source>File Operations</source>
-        <translation>Fájlkezelések</translation>
+        <translation>Fájlműveletek</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="737"/>
         <source>Confirm before deleting files</source>
-        <translation>Törlés előtti megerősítés</translation>
+        <translation>Törlés előtt megerősítés</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="738"/>
         <source>Move deleted files to &quot;trash bin&quot; instead of erasing from disk.</source>
-        <translation>Törölt fájlok lemezről törlés helyett a kukába  a kukába mennek.</translation>
+        <translation>Törölt fájlok lemezről törlés helyett a kukába mennek.</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="783"/>
@@ -1118,111 +1088,112 @@
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="713"/>
         <source>Display</source>
-        <translation type="unfinished"></translation>
+        <translation>Képernyő</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="729"/>
         <source>Bookmarks:</source>
-        <translation type="unfinished"></translation>
+        <translation>Könyvjelzők:</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="732"/>
         <source>Open in current tab</source>
-        <translation type="unfinished"></translation>
+        <translation>Megnyitás itt</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="733"/>
         <source>Open in new tab</source>
-        <translation type="unfinished"></translation>
+        <translation>Megnyitás új fülön</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="734"/>
         <source>Open in new window</source>
-        <translation type="unfinished"></translation>
+        <translation>Megnyitás új ablakban</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="739"/>
         <source>Erase files on removable media instead of &quot;trash can&quot; creation</source>
-        <translation type="unfinished"></translation>
+        <translation>Cserélhető adathordozón a fájlok törlése &quot;kukázás&quot; helyett</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="740"/>
         <source>Confirm before moving files into &quot;trash can&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>Megerősítés &quot;kukázás&quot; előtt</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="741"/>
         <source>Don&apos;t ask options on launch executable file</source>
-        <translation type="unfinished"></translation>
+        <translation>Fájl futtatás előtt nincs opció kérés</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="748"/>
         <source>User interface</source>
-        <translation type="unfinished"></translation>
+        <translation>Kezelőfelület</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="750"/>
         <source>Treat backup files as hidden</source>
-        <translation type="unfinished"></translation>
+        <translation>Biztonsági mentett fájlokat rejtettként kezel</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="751"/>
         <source>Always show full file names</source>
-        <translation type="unfinished"></translation>
+        <translation>Mindig részletes fájlnevek</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="752"/>
         <source>Show icons of hidden files shadowed</source>
-        <translation type="unfinished"></translation>
+        <translation>Rejtett fájlok ikonjai árnyékoltak</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="759"/>
         <source>Show in places</source>
-        <translation type="unfinished"></translation>
+        <translation>Helyben látható</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="764"/>
         <source>Home</source>
-        <translation type="unfinished">Saját könyvtár</translation>
+        <translation>Saját könyvtár</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="766"/>
         <source>Desktop</source>
-        <translation type="unfinished">Asztal</translation>
+        <translation>Asztal</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="768"/>
         <source>Trash can</source>
-        <translation type="unfinished"></translation>
+        <translation>Kuka</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="770"/>
         <source>Computer</source>
-        <translation type="unfinished"></translation>
+        <translation>Számítógép</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="772"/>
         <source>Applications</source>
-        <translation type="unfinished"></translation>
+        <translation>Alkalmazások</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="774"/>
         <source>Devices</source>
-        <translation type="unfinished"></translation>
+        <translation>Eszközök</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="776"/>
         <source>Network</source>
-        <translation type="unfinished"></translation>
+        <translation>Hálózat</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="780"/>
         <source>Do not generate thumbnails for image files exceeding this size:</source>
-        <translation>Nincs, ha a fájl ennél nagyobb:</translation>
+        <translation>Nincs kép, ha a fájl ennél nagyobb:</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="781"/>
         <source> KB</source>
+        <translatorcomment>Jó</translatorcomment>
         <translation></translation>
     </message>
     <message>
@@ -1253,80 +1224,64 @@
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="789"/>
         <source>Close &amp;tab containing removable medium</source>
-        <translation type="unfinished"></translation>
+        <translation>Cserélhe&amp;tő adathortozós fül bezárása</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="790"/>
         <source>Chan&amp;ge folder in the tab to home folder</source>
-        <translation type="unfinished"></translation>
+        <translation>A &amp;fül a saját mappát fogja mutatni</translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_preferences.h" line="791"/>
+        <source>Programs</source>
+        <translation>Programok</translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_preferences.h" line="792"/>
+        <source>Terminal emulator:</source>
+        <translation>Terminálemulátor:</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="793"/>
         <source>Switch &amp;user command:</source>
-        <translation type="unfinished"></translation>
+        <translation>Fel&amp;használó váltási parancs:</translation>
+    </message>
+    <message>
+        <location filename="../../build/pcmanfm/ui_preferences.h" line="794"/>
+        <source>Examples: &quot;xterm -e %s&quot; for terminal or &quot;gksu %s&quot; for switching user.
+%s = the command line you want to execute with terminal or su.</source>
+        <translation>Példa: &quot;xterm -e %s&quot; terminálhoz &quot;gksu %s&quot; felhasználóváltáshoz.
+%s = A parancs, mely terminálhoz, vagy rendszergazdakénti végrehajtáshoz kell.</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="796"/>
         <source>Archiver in&amp;tegration:</source>
-        <translation type="unfinished"></translation>
+        <translation>Csomagoló beépí&amp;tés:</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="797"/>
         <source>Templates</source>
-        <translation type="unfinished"></translation>
+        <translation>Társítások</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="798"/>
         <source>Show only user defined templates in menu</source>
-        <translation type="unfinished"></translation>
+        <translation>Menüben csak felhasználói társítások</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="799"/>
         <source>Show only one template for each MIME type</source>
-        <translation type="unfinished"></translation>
+        <translation>Minden MIME típushoz csak egy társítás látható</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="800"/>
         <source>Run default application after creation from template</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <source>Close tab containing removable medium</source>
-        <translation type="vanished">Cserélhető adathordozó fülének bezárása</translation>
-    </message>
-    <message>
-        <source>Change folder in the tab to home folder</source>
-        <translation type="vanished">A fül a saját mappát fogja mutatni</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_preferences.h" line="791"/>
-        <source>Programs</source>
-        <translation>Programok</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_preferences.h" line="792"/>
-        <source>Terminal emulator:</source>
-        <translation>Terminálemulátor:</translation>
-    </message>
-    <message>
-        <source>Switch user command:</source>
-        <translation type="vanished">Felhasználó váltási parancs:</translation>
-    </message>
-    <message>
-        <location filename="../../build/pcmanfm/ui_preferences.h" line="794"/>
-        <source>Examples: &quot;xterm -e %s&quot; for terminal or &quot;gksu %s&quot; for switching user.
-%s = the command line you want to execute with terminal or su.</source>
-        <translation>Példa: &quot;xterm -e %s&quot; terminálhoz &quot;gksu %s&quot; felhasználóváltáshoz.
-%s = A parancs, mely terminálhoz, vagy rendszergazdakénti végrehajtáshoz kell.</translation>
-    </message>
-    <message>
-        <source>Archiver integration:</source>
-        <translation type="vanished">Csomagoló beépítés:</translation>
+        <translation>Tásítás után az alapértelmezett alkalmazás fut</translation>
     </message>
     <message>
         <location filename="../../build/pcmanfm/ui_preferences.h" line="749"/>
         <source>Use SI decimal prefixes instead of IEC binary prefixes</source>
-        <translation></translation>
+        <translation>IEC bináris helyett SI decimális prefix használata</translation>
     </message>
 </context>
 </TS>
