diff -Naur orig/config/CMakeLists.txt patched/config/CMakeLists.txt
--- orig/config/CMakeLists.txt	2015-05-23 16:26:30.000000000 +0200
+++ patched/config/CMakeLists.txt	2015-05-26 00:39:42.814396508 +0200
@@ -1,9 +1,9 @@
 # install default config files to /etc/xdg
-install(FILES lxqt.conf	DESTINATION ${LXQT_ETC_XDG_DIR}/lxqt/)
-install(FILES session.conf	DESTINATION ${LXQT_ETC_XDG_DIR}/lxqt/)
-install(FILES windowmanagers.conf	DESTINATION ${LXQT_ETC_XDG_DIR}/lxqt/)
+install(FILES lxqt.conf	DESTINATION /etc/xdg/lxqt/)
+install(FILES session.conf	DESTINATION /etc/xdg/lxqt/)
+install(FILES windowmanagers.conf	DESTINATION /etc/xdg/lxqt/)
 
 install(DIRECTORY openbox    DESTINATION ${LXQT_SHARE_DIR}/)
 
 configure_file(pcmanfm-qt/lxqt/settings.conf.in pcmanfm-qt/lxqt/settings.conf @ONLY)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/pcmanfm-qt/lxqt/settings.conf    DESTINATION ${LXQT_ETC_XDG_DIR}/pcmanfm-qt/lxqt/)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/pcmanfm-qt/lxqt/settings.conf    DESTINATION /etc/xdg/pcmanfm-qt/lxqt/)
diff -Naur orig/menu/CMakeLists.txt patched/menu/CMakeLists.txt
--- orig/menu/CMakeLists.txt	2015-05-23 16:26:30.000000000 +0200
+++ patched/menu/CMakeLists.txt	2015-05-26 00:40:08.574524245 +0200
@@ -9,6 +9,6 @@
 #************************************************
 
 install(FILES ${DIRECTORY_FILES} DESTINATION share/desktop-directories)
-install(FILES lxqt-applications.menu	DESTINATION ${LXQT_ETC_XDG_DIR}/menus/)
+install(FILES lxqt-applications.menu	DESTINATION /etc/xdg/menus/)
 
 
