diff -Naur orig/src/cinnamon-screensaver.desktop.in mod/src/cinnamon-screensaver.desktop.in
--- orig/src/cinnamon-screensaver.desktop.in	2014-10-31 16:05:50.000000000 +0100
+++ mod/src/cinnamon-screensaver.desktop.in	2014-11-06 22:34:48.107235350 +0100
@@ -4,7 +4,7 @@
 _Comment=Launch screensaver and locker program
 Icon=preferences-desktop-screensaver
 Exec=cinnamon-screensaver
-OnlyShowIn=X-Cinnamon
+OnlyShowIn=X-Cinnamon;
 NoDisplay=true
 X-GNOME-Autostart-Delay=20
 X-GNOME-Autostart-Phase=Application
diff -Naur orig/src/cinnamon-screensaver.desktop.in.in mod/src/cinnamon-screensaver.desktop.in.in
--- orig/src/cinnamon-screensaver.desktop.in.in	2014-10-31 16:05:50.000000000 +0100
+++ mod/src/cinnamon-screensaver.desktop.in.in	2014-11-06 22:34:59.267361810 +0100
@@ -4,7 +4,7 @@
 _Comment=Launch screensaver and locker program
 Icon=preferences-desktop-screensaver
 Exec=cinnamon-screensaver
-OnlyShowIn=X-Cinnamon
+OnlyShowIn=X-Cinnamon;
 NoDisplay=true
 X-GNOME-Autostart-Delay=20
 X-GNOME-Autostart-Phase=Application
