diff -Naur orig/src/sysinfo.cpp patched/src/sysinfo.cpp
--- orig/src/sysinfo.cpp	2015-07-02 23:46:27.000000000 +0200
+++ patched/src/sysinfo.cpp	2015-09-12 20:35:58.323267745 +0200
@@ -721,13 +721,17 @@
 
     gchar *markup;
 
-
     hbox = gtk_hbox_new(FALSE, 12);
     gtk_container_set_border_width(GTK_CONTAINER(hbox), 6);
 
     /* left-side logo */
 
-    logo = gtk_image_new_from_file(DATADIR "/pixmaps/mate-system-monitor/side.png");
+    if (g_file_test (DATADIR "/pixmaps/mate-system-monitor/distribution/side.png", G_FILE_TEST_EXISTS)) {
+        logo = gtk_image_new_from_file(DATADIR "/pixmaps/mate-system-monitor/distribution/side.png");
+    }
+    else {
+        logo = gtk_image_new_from_file(DATADIR "/pixmaps/mate-system-monitor/side.png");
+    }
     gtk_misc_set_alignment(GTK_MISC(logo), 0.5, 0.0);
     gtk_misc_set_padding(GTK_MISC(logo), 5, 12);
     gtk_box_pack_start(GTK_BOX(hbox), logo, FALSE, FALSE, 0);
