diff -Naur orig/src/sysinfo.cpp patched/src/sysinfo.cpp
--- orig/src/sysinfo.cpp	2015-07-02 23:46:27.000000000 +0200
+++ patched/src/sysinfo.cpp	2015-09-12 23:10:34.309264875 +0200
@@ -569,13 +569,13 @@
 
     SysInfo* get_sysinfo()
     {
-        if (g_file_test ("/etc/os-release", G_FILE_TEST_EXISTS)) {
-            return new GenericSysInfo;
-        }
-        else if (char *p = g_find_program_in_path("lsb_release")) {
+		if (char *p = g_find_program_in_path("lsb_release")) {
             g_free(p);
             return new LSBSysInfo;
         }
+        else  if (g_file_test ("/etc/os-release", G_FILE_TEST_EXISTS)) {
+            return new GenericSysInfo;
+        }
         else if (SysInfo::system() == "SunOS") {
             return new SolarisSysInfo;
         }
