diff -Naur orig/data/lightdm.conf mod/data/lightdm.conf
--- orig/data/lightdm.conf	2014-03-12 20:56:34.000000000 +0100
+++ mod/data/lightdm.conf	2014-09-16 08:11:42.409720058 +0200
@@ -24,7 +24,7 @@
 #user-authority-in-system-dir=false
 #guest-account-script=guest-account
 #log-directory=/var/log/lightdm
-#run-directory=/var/run/lightdm
+run-directory=/run/lightdm
 #cache-directory=/var/cache/lightdm
 #sessions-directory=/usr/share/lightdm/sessions:/usr/share/xsessions
 #remote-sessions-directory=/usr/share/lightdm/remote-sessions
@@ -93,7 +93,7 @@
 #user-session=default
 #allow-guest=true
 #guest-session=
-#session-wrapper=lightdm-session
+session-wrapper=/etc/lightdm/Xsession
 #greeter-wrapper=
 #guest-wrapper=
 #display-setup-script=
@@ -107,6 +107,7 @@
 #autologin-in-background=false
 #autologin-session=UNIMPLEMENTED
 #exit-on-failure=false
+pam-service=lightdm-autologin
 
 #
 # Seat configuration
