diff -Nur orig/source/gameengine/VideoTexture/VideoFFmpeg.cpp mod/source/gameengine/VideoTexture/VideoFFmpeg.cpp
--- orig/source/gameengine/VideoTexture/VideoFFmpeg.cpp	2017-01-18 21:14:52.000000000 +0100
+++ mod/source/gameengine/VideoTexture/VideoFFmpeg.cpp	2017-01-18 21:21:33.639203854 +0100
@@ -390,18 +390,6 @@
 					if (   input->data[0]!=0 || input->data[1]!=0 
 						|| input->data[2]!=0 || input->data[3]!=0)
 					{
-						if (video->m_deinterlace) 
-						{
-							if (avpicture_deinterlace(
-								(AVPicture*) video->m_frameDeinterlaced,
-								(const AVPicture*) video->m_frame,
-								video->m_codecCtx->pix_fmt,
-								video->m_codecCtx->width,
-								video->m_codecCtx->height) >= 0)
-							{
-								input = video->m_frameDeinterlaced;
-							}
-						}
 						// convert to RGB24
 						sws_scale(video->m_imgConvertCtx,
 							input->data,
@@ -1056,17 +1044,7 @@
 				}
 
 				if (m_deinterlace) 
-				{
-					if (avpicture_deinterlace(
-						(AVPicture*) m_frameDeinterlaced,
-						(const AVPicture*) m_frame,
-						m_codecCtx->pix_fmt,
-						m_codecCtx->width,
-						m_codecCtx->height) >= 0)
-					{
-						input = m_frameDeinterlaced;
-					}
-				}
+
 				// convert to RGB24
 				sws_scale(m_imgConvertCtx,
 					input->data,
