diff -Naur ORIG/configure.py UHU/configure.py
--- ORIG/configure.py	2014-09-12 23:58:40.000000000 +0200
+++ UHU/configure.py	2014-10-01 22:03:58.183377476 +0200
@@ -87,7 +87,7 @@
     'QtSvg':                ModuleMetadata(qmake_QT=['svg']),
     'QtTest':               ModuleMetadata(qmake_QT=['testlib', 'widgets']),
     'QtWebKit':             ModuleMetadata(qmake_QT=['webkit', 'network']),
-    'QtWebKitWidgets':      ModuleMetadata(qmake_QT=['webkitwidgets']),
+    'QtWebKitWidgets':      ModuleMetadata(qmake_QT=['webkitwidgets', 'printsupport']),
     'QtWebSockets':         ModuleMetadata(qmake_QT=['websockets']),
     'QtWidgets':            ModuleMetadata(qmake_QT=['widgets']),
     'QtWinExtras':          ModuleMetadata(qmake_QT=['winextras', 'widgets']),
