diff -Naur orig/content/browser/time_zone_monitor.cc patched/content/browser/time_zone_monitor.cc
--- orig/content/browser/time_zone_monitor.cc	2015-08-22 21:01:55.000000000 +0200
+++ patched/content/browser/time_zone_monitor.cc	2015-09-23 15:00:24.931976412 +0200
@@ -22,9 +22,9 @@
 
 void TimeZoneMonitor::NotifyRenderers() {
   DCHECK_CURRENTLY_ON(BrowserThread::UI);
-#if defined(OS_CHROMEOS)
-  // On CrOS, ICU's default tz is already set to a new zone. No
-  // need to redetect it with detectHostTimeZone().
+#if defined(OS_CHROMEOS) || U_ICU_VERSION_MAJOR_NUM < 54
+  // On CrOS and older ICU versions, ICU's default tz is already set to a new zone.
+  // No need to redetect it with detectHostTimeZone().
   scoped_ptr<icu::TimeZone> new_zone(icu::TimeZone::createDefault());
 #else
   icu::TimeZone* new_zone = icu::TimeZone::detectHostTimeZone();
