diff -urN mariadb-10.0.11.orig/packaging/rpm-oel/mysqld.service mariadb-10.0.11/packaging/rpm-oel/mysqld.service
--- mariadb-10.0.11.orig/packaging/rpm-oel/mysqld.service	2014-05-11 00:36:18.000000000 +0200
+++ mariadb-10.0.11/packaging/rpm-oel/mysqld.service	2014-05-27 10:46:42.000000000 +0200
@@ -30,19 +30,14 @@
 # Execute pre and post scripts as root
 PermissionsStartOnly=true
 
-# Needed to create system tables etc.
-ExecStartPre=/usr/bin/mysql-systemd-start pre
-
 # Start main service
 ExecStart=/usr/bin/mysqld_safe
 
 # Don't signal startup success before a ping works
-ExecStartPost=/usr/bin/mysql-systemd-start post
+ExecStartPost=/usr/bin/mysqld-ping
 
 # Give up if ping don't get an answer
 TimeoutSec=600
 
 Restart=always
 PrivateTmp=false
-
-
