diff -Naur orig/packaging/rpm-oel/mysqld.service patched/packaging/rpm-oel/mysqld.service
--- orig/packaging/rpm-oel/mysqld.service	2015-06-17 16:54:12.000000000 +0200
+++ patched/packaging/rpm-oel/mysqld.service	2015-08-02 21:33:39.816801937 +0200
@@ -17,7 +17,6 @@
 [Unit]
 Description=MySQL Community Server
 After=network.target
-After=syslog.target
 
 [Install]
 WantedBy=multi-user.target
@@ -30,14 +29,11 @@
 # Execute pre and post scripts as root
 PermissionsStartOnly=true
 
-# Needed to create system tables etc.
-ExecStartPre=/usr/bin/mysql-systemd-start pre
-
 # Start main service
 ExecStart=/usr/bin/mysqld_safe
 
 # Don't signal startup success before a ping works
-ExecStartPost=/usr/bin/mysql-systemd-start post
+ExecStartPost=/usr/bin/mysqld-ping
 
 # Give up if ping don't get an answer
 TimeoutSec=600
@@ -45,4 +41,3 @@
 Restart=always
 PrivateTmp=false
 
-
