diff -Naur orig/configure patched/configure
--- orig/configure	2014-07-15 19:16:46.000000000 +0200
+++ patched/configure	2014-08-30 15:10:54.449082053 +0200
@@ -2383,6 +2383,9 @@
     --target-arch=*)
       TARGET_MACHINE=`echo $option | cut -d'=' -f2`
       ;;
+	--target-cpu=*)
+      TARGET_CPU=`echo $option | cut -d'=' -f2`
+      ;;
     --disable-xpcom)
       [ $WITH_XPCOM -eq 1 ] && WITH_XPCOM=0
       ;;
