diff -Naur orig/operations/external/ff-load.c patched/operations/external/ff-load.c
--- orig/operations/external/ff-load.c	2015-05-17 02:32:23.000000000 +0200
+++ patched/operations/external/ff-load.c	2015-06-08 23:12:09.125947117 +0200
@@ -325,7 +325,7 @@
       p->width = p->enc->width;
       p->height = p->enc->height;
       p->frames = 10000000;
-      p->lavc_frame = avcodec_alloc_frame ();
+      p->lavc_frame = av_frame_alloc ();
 
       if (p->fourcc)
         g_free (p->fourcc);
diff -Naur orig/operations/workshop/external/ff-save.c patched/operations/workshop/external/ff-save.c
--- orig/operations/workshop/external/ff-save.c	2015-05-24 20:22:08.000000000 +0200
+++ patched/operations/workshop/external/ff-save.c	2015-06-08 23:07:16.416495649 +0200
@@ -573,7 +573,7 @@
     }
 
   /* open the codec */
-  if (avcodec_open (c, codec) < 0)
+  if (avcodec_open2 (c, codec, NULL) < 0)
     {
       fprintf (stderr, "could not open codec\n");
       exit (1);
@@ -774,26 +774,20 @@
      /*XXX: FOO p->audio_st = add_audio_stream (op, p->oc, p->fmt->audio_codec);*/
     }
 
-  if (av_set_parameters (p->oc, NULL) < 0)
-    {
-      fprintf (stderr, "Invalid output format propeters\n%s", "");
-      return -1;
-    }
-
-  dump_format (p->oc, 0, self->path, 1);
+  av_dump_format (p->oc, 0, self->path, 1);
 
   if (p->video_st)
     open_video (p, p->oc, p->video_st);
   if (p->audio_st)
     open_audio (p, p->oc, p->audio_st);
 
-  if (url_fopen (&p->oc->pb, self->path, URL_WRONLY) < 0)
+  if (avio_open (&p->oc->pb, self->path, AVIO_FLAG_WRITE) < 0)
     {
       fprintf (stderr, "couldn't open '%s'\n", self->path);
       return -1;
     }
 
-  av_write_header (p->oc);
+  avformat_write_header (p->oc, NULL);
 
   return 0;
 }
@@ -863,7 +857,7 @@
             av_freep (&p->oc->streams[i]);
           }
 
-        url_fclose (&p->oc->pb);
+        avio_close (&p->oc->pb);
         free (p->oc);
       }
       g_free (o->user_data);
