--- a/src/core/window.c	2015-11-09 10:59:23.000000000 +0000
+++ b/src/core/window.c	2015-11-09 18:32:14.950289767 +0000
@@ -10709,7 +10709,7 @@ find_ancestor_func (MetaWindow *window,
  * so by traversing the @transient's ancestors until it either locates @window
  * or reaches an ancestor that is not transient.
  *
- * Return Value: (transfer none): %TRUE if window is an ancestor of transient.
+ * Return Value: %TRUE if window is an ancestor of transient.
  */
 gboolean
 meta_window_is_ancestor_of_transient (MetaWindow *window,
@@ -11549,7 +11549,7 @@ meta_window_get_transient_for (MetaWindo
  * parents. A typical usage of this hint is for a dialog that wants to stay
  * above its associated window.
  *
- * Return value: (transfer none): the window this window is transient for, or
+ * Return value: the window this window is transient for, or
  * None if the WM_TRANSIENT_FOR hint is unset.
  */
 Window
@@ -11565,7 +11565,7 @@ meta_window_get_transient_for_as_xid (Me
  * Returns pid of the process that created this window, if known (obtained from
  * the _NET_WM_PID property).
  *
- * Return value: (transfer none): the pid, or -1 if not known.
+ * Return value: the pid, or -1 if not known.
  */
 int
 meta_window_get_pid (MetaWindow *window)
@@ -11618,7 +11618,7 @@ meta_window_is_remote (MetaWindow *windo
  * Queries whether the window is in a modal state as described by the
  * _NET_WM_STATE protocol.
  *
- * Return value: (transfer none): TRUE if the window is in modal state.
+ * Return value: TRUE if the window is in modal state.
  */
 gboolean
 meta_window_is_modal (MetaWindow *window)
--- a/src/core/display.c	2015-11-09 10:59:23.000000000 +0000
+++ b/src/core/display.c	2015-11-09 19:43:22.983370663 +0000
@@ -5595,7 +5595,7 @@ Atom meta_display_get_atom (MetaDisplay
  * _NET_SUPPORTING_WM_CHECK mechanism of EWMH). For use by plugins that wish
  * to attach additional custom properties to this window.
  *
- * Return value: (transfer none): xid of the leader window.
+ * Return value: xid of the leader window.
  **/
 Window
 meta_display_get_leader_window (MetaDisplay *display)
--- a/src/compositor/meta-window-actor.c	2015-11-09 10:59:23.000000000 +0000
+++ b/src/compositor/meta-window-actor.c	2015-11-09 19:46:12.717281189 +0000
@@ -963,7 +963,7 @@ const char *meta_window_actor_get_descri
  * This function is deprecated  and should not be used in newly written code;
  * meta_window_get_workspace() instead.
  *
- * Return value: (transfer none): index of workspace on which this window is
+ * Return value: index of workspace on which this window is
  * located.
  */
 gint
