diff -Naur orig/clutter/clutter-actor.c patched/clutter/clutter-actor.c
--- orig/clutter/clutter-actor.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-actor.c	2015-07-19 15:46:45.159814371 +0200
@@ -6647,7 +6647,7 @@
  * Return value: the name of the actor, or %NULL. The returned string is
  *   owned by the actor and should not be modified or freed.
  */
-G_CONST_RETURN gchar *
+const gchar *
 clutter_actor_get_name (ClutterActor *self)
 {
   g_return_val_if_fail (CLUTTER_IS_ACTOR (self), NULL);
diff -Naur orig/clutter/clutter-actor.h patched/clutter/clutter-actor.h
--- orig/clutter/clutter-actor.h	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-actor.h	2015-07-19 15:45:47.939530631 +0200
@@ -426,7 +426,7 @@
 
 void                  clutter_actor_set_name                  (ClutterActor          *self,
                                                                const gchar           *name);
-G_CONST_RETURN gchar *clutter_actor_get_name                  (ClutterActor          *self);
+const gchar *clutter_actor_get_name                  (ClutterActor          *self);
 
 guint32               clutter_actor_get_gid                   (ClutterActor          *self);
 void                  clutter_actor_set_clip                  (ClutterActor          *self,
diff -Naur orig/clutter/clutter-animator.c patched/clutter/clutter-animator.c
--- orig/clutter/clutter-animator.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-animator.c	2015-07-19 15:47:46.012116122 +0200
@@ -2025,7 +2025,7 @@
  *
  * Since: 1.2
  */
-G_CONST_RETURN gchar *
+const gchar *
 clutter_animator_key_get_property_name (const ClutterAnimatorKey *key)
 {
   g_return_val_if_fail (key != NULL, NULL);
diff -Naur orig/clutter/clutter-animator.h patched/clutter/clutter-animator.h
--- orig/clutter/clutter-animator.h	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-animator.h	2015-07-19 15:44:56.283274482 +0200
@@ -161,7 +161,7 @@
 
 GType                 clutter_animator_key_get_type          (void) G_GNUC_CONST;
 GObject *             clutter_animator_key_get_object        (const ClutterAnimatorKey *key);
-G_CONST_RETURN gchar *clutter_animator_key_get_property_name (const ClutterAnimatorKey *key);
+const gchar *clutter_animator_key_get_property_name (const ClutterAnimatorKey *key);
 GType                 clutter_animator_key_get_property_type (const ClutterAnimatorKey *key);
 gulong                clutter_animator_key_get_mode          (const ClutterAnimatorKey *key);
 gdouble               clutter_animator_key_get_progress      (const ClutterAnimatorKey *key);
diff -Naur orig/clutter/clutter-backend.c patched/clutter/clutter-backend.c
--- orig/clutter/clutter-backend.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-backend.c	2015-07-19 15:45:39.415488363 +0200
@@ -775,7 +775,7 @@
  *
  * Since: 1.0
  */
-G_CONST_RETURN gchar *
+const gchar *
 clutter_backend_get_font_name (ClutterBackend *backend)
 {
   ClutterBackendPrivate *priv;
diff -Naur orig/clutter/clutter-backend.h patched/clutter/clutter-backend.h
--- orig/clutter/clutter-backend.h	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-backend.h	2015-07-19 15:45:32.067451926 +0200
@@ -108,7 +108,7 @@
 const cairo_font_options_t *clutter_backend_get_font_options          (ClutterBackend             *backend);
 void                        clutter_backend_set_font_name             (ClutterBackend             *backend,
                                                                        const gchar                *font_name);
-G_CONST_RETURN gchar *      clutter_backend_get_font_name             (ClutterBackend             *backend);
+const gchar *      clutter_backend_get_font_name             (ClutterBackend             *backend);
 
 G_END_DECLS
 
diff -Naur orig/clutter/clutter-binding-pool.c patched/clutter/clutter-binding-pool.c
--- orig/clutter/clutter-binding-pool.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-binding-pool.c	2015-07-19 15:47:51.476143216 +0200
@@ -717,7 +717,7 @@
  *
  * Since: 1.0
  */
-G_CONST_RETURN gchar *
+const gchar *
 clutter_binding_pool_find_action (ClutterBindingPool  *pool,
                                   guint                key_val,
                                   ClutterModifierType  modifiers)
diff -Naur orig/clutter/clutter-binding-pool.h patched/clutter/clutter-binding-pool.h
--- orig/clutter/clutter-binding-pool.h	2010-08-21 18:46:16.000000000 +0200
+++ patched/clutter/clutter-binding-pool.h	2015-07-19 15:48:03.556203118 +0200
@@ -97,7 +97,7 @@
                                                              ClutterModifierType  modifiers,
                                                              GClosure            *closure);
 
-G_CONST_RETURN gchar *clutter_binding_pool_find_action      (ClutterBindingPool  *pool,
+const gchar *clutter_binding_pool_find_action      (ClutterBindingPool  *pool,
                                                              guint                key_val,
                                                              ClutterModifierType  modifiers);
 void                  clutter_binding_pool_remove_action    (ClutterBindingPool  *pool,
diff -Naur orig/clutter/clutter-color.c patched/clutter/clutter-color.c
--- orig/clutter/clutter-color.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-color.c	2015-07-19 15:47:24.628010084 +0200
@@ -768,7 +768,7 @@
  *
  * Since: 0.8.4
  */
-G_CONST_RETURN ClutterColor *
+const ClutterColor *
 clutter_value_get_color (const GValue *value)
 {
   g_return_val_if_fail (CLUTTER_VALUE_HOLDS_COLOR (value), NULL);
diff -Naur orig/clutter/clutter-color.h patched/clutter/clutter-color.h
--- orig/clutter/clutter-color.h	2010-08-21 18:46:16.000000000 +0200
+++ patched/clutter/clutter-color.h	2015-07-19 15:46:59.431885143 +0200
@@ -138,7 +138,7 @@
 
 void                         clutter_value_set_color (GValue             *value,
                                                       const ClutterColor *color);
-G_CONST_RETURN ClutterColor *clutter_value_get_color (const GValue       *value);
+const ClutterColor *clutter_value_get_color (const GValue       *value);
 
 GType       clutter_param_color_get_type (void) G_GNUC_CONST;
 GParamSpec *clutter_param_spec_color     (const gchar        *name,
diff -Naur orig/clutter/clutter-event.c patched/clutter/clutter-event.c
--- orig/clutter/clutter-event.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-event.c	2015-07-19 15:46:52.819852355 +0200
@@ -782,7 +782,7 @@
  *
  * Since: 1.2
  */
-G_CONST_RETURN ClutterEvent *
+const ClutterEvent *
 clutter_get_current_event (void)
 {
   ClutterMainContext *context = _clutter_context_get_default ();
diff -Naur orig/clutter/clutter-event.h patched/clutter/clutter-event.h
--- orig/clutter/clutter-event.h	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-event.h	2015-07-19 15:44:44.707217079 +0200
@@ -475,7 +475,7 @@
 guint32                clutter_keysym_to_unicode            (guint             keyval);
 
 guint32                clutter_get_current_event_time       (void);
-G_CONST_RETURN ClutterEvent *clutter_get_current_event      (void);
+const ClutterEvent *clutter_get_current_event      (void);
 
 G_END_DECLS
 
diff -Naur orig/clutter/clutter-input-device.c patched/clutter/clutter-input-device.c
--- orig/clutter/clutter-input-device.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-input-device.c	2015-07-19 15:46:31.903748638 +0200
@@ -586,7 +586,7 @@
  *
  * Since: 1.2
  */
-G_CONST_RETURN gchar *
+const gchar *
 clutter_input_device_get_device_name (ClutterInputDevice *device)
 {
   g_return_val_if_fail (CLUTTER_IS_INPUT_DEVICE (device), NULL);
diff -Naur orig/clutter/clutter-input-device.h patched/clutter/clutter-input-device.h
--- orig/clutter/clutter-input-device.h	2010-08-21 18:46:16.000000000 +0200
+++ patched/clutter/clutter-input-device.h	2015-07-19 15:47:18.455979478 +0200
@@ -93,7 +93,7 @@
                                                                gint               *y);
 ClutterActor *         clutter_input_device_get_pointer_actor (ClutterInputDevice *device);
 ClutterStage *         clutter_input_device_get_pointer_stage (ClutterInputDevice *device);
-G_CONST_RETURN gchar * clutter_input_device_get_device_name   (ClutterInputDevice *device);
+const gchar * clutter_input_device_get_device_name   (ClutterInputDevice *device);
 
 void                   clutter_input_device_update_from_event (ClutterInputDevice *device,
                                                                ClutterEvent       *event,
diff -Naur orig/clutter/clutter-main.c patched/clutter/clutter-main.c
--- orig/clutter/clutter-main.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-main.c	2015-07-19 16:04:27.217080829 +0200
@@ -945,9 +945,6 @@
 void
 clutter_threads_init (void)
 {
-  if (!g_thread_supported ())
-    g_error ("g_thread_init() must be called before clutter_threads_init()");
-
   if (clutter_threads_mutex != NULL)
     return;
 
diff -Naur orig/clutter/clutter-model.c patched/clutter/clutter-model.c
--- orig/clutter/clutter-model.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-model.c	2015-07-19 15:45:56.047570837 +0200
@@ -1215,7 +1215,7 @@
  *
  * Since: 0.6
  */
-G_CONST_RETURN gchar *
+const gchar *
 clutter_model_get_column_name (ClutterModel *model,
                                guint         column)
 {
diff -Naur orig/clutter/clutter-model.h patched/clutter/clutter-model.h
--- orig/clutter/clutter-model.h	2010-08-21 18:46:16.000000000 +0200
+++ patched/clutter/clutter-model.h	2015-07-19 15:44:34.675167333 +0200
@@ -226,7 +226,7 @@
 
 guint                 clutter_model_get_n_rows         (ClutterModel     *model);
 guint                 clutter_model_get_n_columns      (ClutterModel     *model);
-G_CONST_RETURN gchar *clutter_model_get_column_name    (ClutterModel     *model,
+const gchar *clutter_model_get_column_name    (ClutterModel     *model,
                                                         guint             column);
 GType                 clutter_model_get_column_type    (ClutterModel     *model,
                                                         guint             column);
diff -Naur orig/clutter/clutter-scriptable.c patched/clutter/clutter-scriptable.c
--- orig/clutter/clutter-scriptable.c	2010-08-21 18:46:16.000000000 +0200
+++ patched/clutter/clutter-scriptable.c	2015-07-19 15:46:38.299780354 +0200
@@ -112,7 +112,7 @@
  *
  * Since: 0.6
  */
-G_CONST_RETURN gchar *
+const gchar *
 clutter_scriptable_get_id (ClutterScriptable *scriptable)
 {
   ClutterScriptableIface *iface;
diff -Naur orig/clutter/clutter-scriptable.h patched/clutter/clutter-scriptable.h
--- orig/clutter/clutter-scriptable.h	2010-09-23 16:51:34.000000000 +0200
+++ patched/clutter/clutter-scriptable.h	2015-07-19 15:46:16.951674495 +0200
@@ -91,7 +91,7 @@
 
 void                  clutter_scriptable_set_id              (ClutterScriptable *scriptable,
                                                               const gchar       *id);
-G_CONST_RETURN gchar *clutter_scriptable_get_id              (ClutterScriptable *scriptable);
+const gchar *clutter_scriptable_get_id              (ClutterScriptable *scriptable);
 gboolean              clutter_scriptable_parse_custom_node   (ClutterScriptable *scriptable,
                                                               ClutterScript     *script,
                                                               GValue            *value,
diff -Naur orig/clutter/clutter-script.c patched/clutter/clutter-script.c
--- orig/clutter/clutter-script.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-script.c	2015-07-19 15:47:05.347914479 +0200
@@ -807,7 +807,7 @@
  *
  * Since: 0.6
  */
-G_CONST_RETURN gchar *
+const gchar *
 clutter_get_script_id (GObject *gobject)
 {
   g_return_val_if_fail (G_IS_OBJECT (gobject), NULL);
diff -Naur orig/clutter/clutter-script.h patched/clutter/clutter-script.h
--- orig/clutter/clutter-script.h	2010-08-21 18:46:16.000000000 +0200
+++ patched/clutter/clutter-script.h	2015-07-19 15:47:39.712084881 +0200
@@ -168,7 +168,7 @@
 GType          clutter_script_get_type_from_name   (ClutterScript  *script,
                                                     const gchar    *type_name);
 
-G_CONST_RETURN gchar *clutter_get_script_id        (GObject        *gobject);
+const gchar *clutter_get_script_id        (GObject        *gobject);
 
 void           clutter_script_connect_signals      (ClutterScript  *script,
                                                     gpointer        user_data);
diff -Naur orig/clutter/clutter-shader.c patched/clutter/clutter-shader.c
--- orig/clutter/clutter-shader.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-shader.c	2015-07-19 15:45:11.911351977 +0200
@@ -794,7 +794,7 @@
  *
  * Since: 0.6
  */
-G_CONST_RETURN gchar *
+const gchar *
 clutter_shader_get_fragment_source (ClutterShader *shader)
 {
   g_return_val_if_fail (CLUTTER_IS_SHADER (shader), NULL);
@@ -814,7 +814,7 @@
  *
  * Since: 0.6
  */
-G_CONST_RETURN gchar *
+const gchar *
 clutter_shader_get_vertex_source (ClutterShader *shader)
 {
   g_return_val_if_fail (CLUTTER_IS_SHADER (shader), NULL);
diff -Naur orig/clutter/clutter-shader.h patched/clutter/clutter-shader.h
--- orig/clutter/clutter-shader.h	2010-08-21 18:46:16.000000000 +0200
+++ patched/clutter/clutter-shader.h	2015-07-19 15:46:07.359626930 +0200
@@ -118,8 +118,8 @@
                                                                const gchar        *data,
                                                                gssize              length);
 
-G_CONST_RETURN gchar *clutter_shader_get_vertex_source        (ClutterShader      *shader);
-G_CONST_RETURN gchar *clutter_shader_get_fragment_source      (ClutterShader      *shader);
+const gchar *clutter_shader_get_vertex_source        (ClutterShader      *shader);
+const gchar *clutter_shader_get_fragment_source      (ClutterShader      *shader);
 
 void                  clutter_shader_set_uniform              (ClutterShader      *shader,
                                                                const gchar        *name,
diff -Naur orig/clutter/clutter-shader-types.c patched/clutter/clutter-shader-types.c
--- orig/clutter/clutter-shader-types.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-shader-types.c	2015-07-19 15:47:33.160052392 +0200
@@ -482,7 +482,7 @@
  *
  * Since: 0.8
  */
-G_CONST_RETURN gfloat *
+const gfloat *
 clutter_value_get_shader_float (const GValue *value,
                                 gsize        *length)
 {
@@ -514,7 +514,7 @@
  *
  * Since: 0.8
  */
-G_CONST_RETURN gint *
+const gint *
 clutter_value_get_shader_int (const GValue *value,
                               gsize        *length)
 {
@@ -546,7 +546,7 @@
  *
  * Since: 0.8
  */
-G_CONST_RETURN gfloat *
+const gfloat *
 clutter_value_get_shader_matrix (const GValue *value,
                                  gsize        *length)
 {
diff -Naur orig/clutter/clutter-shader-types.h patched/clutter/clutter-shader-types.h
--- orig/clutter/clutter-shader-types.h	2010-08-21 18:46:16.000000000 +0200
+++ patched/clutter/clutter-shader-types.h	2015-07-19 15:45:05.099318198 +0200
@@ -84,11 +84,11 @@
 void                    clutter_value_set_shader_matrix (GValue       *value,
                                                          gint          size,
                                                          const gfloat *matrix);
-G_CONST_RETURN gfloat * clutter_value_get_shader_float  (const GValue *value,
+const gfloat * clutter_value_get_shader_float  (const GValue *value,
                                                          gsize        *length);
-G_CONST_RETURN gint   * clutter_value_get_shader_int    (const GValue *value,
+const gint   * clutter_value_get_shader_int    (const GValue *value,
                                                          gsize        *length);
-G_CONST_RETURN gfloat * clutter_value_get_shader_matrix (const GValue *value,
+const gfloat * clutter_value_get_shader_matrix (const GValue *value,
                                                          gsize        *length);
 
 G_END_DECLS
diff -Naur orig/clutter/clutter-stage.c patched/clutter/clutter-stage.c
--- orig/clutter/clutter-stage.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-stage.c	2015-07-19 15:48:18.080275139 +0200
@@ -1821,7 +1821,7 @@
  *
  * Since: 0.4
  **/
-G_CONST_RETURN gchar *
+const gchar *
 clutter_stage_get_title (ClutterStage       *stage)
 {
   g_return_val_if_fail (CLUTTER_IS_STAGE (stage), NULL);
diff -Naur orig/clutter/clutter-stage.h patched/clutter/clutter-stage.h
--- orig/clutter/clutter-stage.h	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-stage.h	2015-07-19 15:44:27.343130975 +0200
@@ -218,7 +218,7 @@
 
 void                  clutter_stage_set_title          (ClutterStage *stage,
                                                         const gchar  *title);
-G_CONST_RETURN gchar *clutter_stage_get_title          (ClutterStage *stage);
+const gchar *clutter_stage_get_title          (ClutterStage *stage);
 void                  clutter_stage_set_user_resizable (ClutterStage *stage,
 						        gboolean      resizable);
 gboolean              clutter_stage_get_user_resizable (ClutterStage *stage);
diff -Naur orig/clutter/clutter-text.c patched/clutter/clutter-text.c
--- orig/clutter/clutter-text.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-text.c	2015-07-19 15:45:24.807415925 +0200
@@ -3718,7 +3718,7 @@
  *
  * Since: 1.0
  */
-G_CONST_RETURN gchar *
+const gchar *
 clutter_text_get_font_name (ClutterText *text)
 {
   g_return_val_if_fail (CLUTTER_IS_TEXT (text), NULL);
@@ -3800,7 +3800,7 @@
  *
  * Since: 1.0
  */
-G_CONST_RETURN gchar *
+const gchar *
 clutter_text_get_text (ClutterText *self)
 {
   g_return_val_if_fail (CLUTTER_IS_TEXT (self), NULL);
diff -Naur orig/clutter/clutter-text.h patched/clutter/clutter-text.h
--- orig/clutter/clutter-text.h	2010-08-21 18:46:16.000000000 +0200
+++ patched/clutter/clutter-text.h	2015-07-19 15:48:10.656238325 +0200
@@ -103,7 +103,7 @@
 ClutterActor *        clutter_text_new_with_text        (const gchar          *font_name,
                                                          const gchar          *text);
 
-G_CONST_RETURN gchar *clutter_text_get_text             (ClutterText          *self);
+const gchar *clutter_text_get_text             (ClutterText          *self);
 void                  clutter_text_set_text             (ClutterText          *self,
                                                          const gchar          *text);
 void                  clutter_text_set_markup           (ClutterText          *self,
@@ -114,7 +114,7 @@
                                                          ClutterColor         *color);
 void                  clutter_text_set_font_name        (ClutterText          *self,
                                                          const gchar          *font_name);
-G_CONST_RETURN gchar *clutter_text_get_font_name        (ClutterText          *self);
+const gchar *clutter_text_get_font_name        (ClutterText          *self);
 void                  clutter_text_set_font_description (ClutterText          *self,
                                                          PangoFontDescription *font_desc);
 PangoFontDescription *clutter_text_get_font_description (ClutterText          *self);
diff -Naur orig/clutter/clutter-texture.c patched/clutter/clutter-texture.c
--- orig/clutter/clutter-texture.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-texture.c	2015-07-19 16:03:59.644944106 +0200
@@ -154,7 +154,7 @@
 static GThreadPool *async_thread_pool = NULL;
 static guint        repaint_upload_func = 0;
 static GList       *upload_list = NULL;
-static GStaticMutex upload_list_mutex = G_STATIC_MUTEX_INIT;
+static GMutex		upload_list_mutex;
 
 static void
 texture_fbo_free_resources (ClutterTexture *texture);
@@ -1536,7 +1536,7 @@
 {
   gulong start_time;
 
-  g_static_mutex_lock (&upload_list_mutex);
+  g_mutex_lock (&upload_list_mutex);
 
   if (upload_list)
     {
@@ -1564,7 +1564,7 @@
       _clutter_master_clock_ensure_next_iteration (master_clock);
     }
 
-  g_static_mutex_unlock (&upload_list_mutex);
+  g_mutex_unlock (&upload_list_mutex);
 
   return TRUE;
 }
@@ -1611,7 +1611,7 @@
        * set it while we're holding the mutex so we can safely start the
        * idle handler now without the possibility of calling the
        * callback after it is aborted */
-      g_static_mutex_lock (&upload_list_mutex);
+      g_mutex_lock (&upload_list_mutex);
 
       if (repaint_upload_func == 0)
         {
@@ -1623,7 +1623,7 @@
       upload_list = g_list_append (upload_list, data);
       data->upload_queued = TRUE;
 
-      g_static_mutex_unlock (&upload_list_mutex);
+      g_mutex_unlock (&upload_list_mutex);
 
       g_mutex_unlock (data->mutex);
 
@@ -1727,25 +1727,10 @@
 
   priv->async_data = data;
 
-  if (g_thread_supported ())
-    {
-      data->mutex = g_mutex_new ();
-
-      if (async_thread_pool == NULL)
-        /* This apparently can't fail if exclusive == FALSE */
-        async_thread_pool
-          = g_thread_pool_new (clutter_texture_thread_func,
-                               NULL, 1, FALSE, NULL);
+  data->mutex = NULL;
 
-      g_thread_pool_push (async_thread_pool, data, NULL);
-    }
-  else
-    {
-      data->mutex = NULL;
-
-      data->load_idle
+  data->load_idle
         = clutter_threads_add_idle (clutter_texture_idle_func, data);
-    }
 
   return TRUE;
 }
diff -Naur orig/clutter/clutter-units.c patched/clutter/clutter-units.c
--- orig/clutter/clutter-units.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/clutter-units.c	2015-07-19 15:47:57.720174179 +0200
@@ -794,7 +794,7 @@
  *
  * Since: 0.8
  */
-G_CONST_RETURN ClutterUnits *
+const ClutterUnits *
 clutter_value_get_units (const GValue *value)
 {
   g_return_val_if_fail (CLUTTER_VALUE_HOLDS_UNITS (value), NULL);
diff -Naur orig/clutter/clutter-units.h patched/clutter/clutter-units.h
--- orig/clutter/clutter-units.h	2010-08-21 18:46:16.000000000 +0200
+++ patched/clutter/clutter-units.h	2015-07-19 15:47:12.723951054 +0200
@@ -181,7 +181,7 @@
 
 void                         clutter_value_set_units  (GValue             *value,
                                                        const ClutterUnits *units);
-G_CONST_RETURN ClutterUnits *clutter_value_get_units  (const GValue       *value);
+const ClutterUnits *clutter_value_get_units  (const GValue       *value);
 
 G_END_DECLS
 
diff -Naur orig/clutter/cogl/cogl/cogl-material.h patched/clutter/cogl/cogl/cogl-material.h
--- orig/clutter/cogl/cogl/cogl-material.h	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/cogl/cogl/cogl-material.h	2015-07-19 15:49:35.808660573 +0200
@@ -735,7 +735,7 @@
  *   functions. The list is owned by COGL and it  should not be modified or
  *   freed
  */
-G_CONST_RETURN GList *
+const GList *
 cogl_material_get_layers (CoglHandle material);
 
 /**
diff -Naur orig/clutter/cogl/cogl/cogl-matrix.h patched/clutter/cogl/cogl/cogl-matrix.h
--- orig/clutter/cogl/cogl/cogl-matrix.h	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/cogl/cogl/cogl-matrix.h	2015-07-19 15:49:42.740694947 +0200
@@ -273,7 +273,7 @@
  *
  * Return value: a pointer to the float array
  */
-G_CONST_RETURN float *
+const float *
 cogl_matrix_get_array (const CoglMatrix *matrix);
 
 /**
diff -Naur orig/clutter/glx/clutter-backend-glx.c patched/clutter/glx/clutter-backend-glx.c
--- orig/clutter/glx/clutter-backend-glx.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/glx/clutter-backend-glx.c	2015-07-19 15:49:16.688565762 +0200
@@ -58,7 +58,7 @@
 
 static gchar    *clutter_vblank_name = NULL;
 
-G_CONST_RETURN gchar*
+const gchar*
 clutter_backend_glx_get_vblank_method (void)
 {
   return clutter_vblank_name;
diff -Naur orig/clutter/json/json-array.c patched/clutter/json/json-array.c
--- orig/clutter/json/json-array.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/json/json-array.c	2015-07-19 15:48:39.736382526 +0200
@@ -332,7 +332,7 @@
  *
  * Since: 0.8
  */
-G_CONST_RETURN gchar *
+const gchar *
 json_array_get_string_element (JsonArray *array,
                                guint      index_)
 {
diff -Naur orig/clutter/json/json-node.c patched/clutter/json/json-node.c
--- orig/clutter/json/json-node.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/json/json-node.c	2015-07-19 15:49:09.616530693 +0200
@@ -461,7 +461,7 @@
  * Return value: a string containing the name of the type. The returned string
  *   is owned by the node and should never be modified or freed
  */
-G_CONST_RETURN gchar *
+const gchar *
 json_node_type_name (JsonNode *node)
 {
   g_return_val_if_fail (node != NULL, "(null)");
@@ -538,7 +538,7 @@
  *
  * Return value: a string value.
  */
-G_CONST_RETURN gchar *
+const gchar *
 json_node_get_string (JsonNode *node)
 {
   g_return_val_if_fail (node != NULL, NULL);
diff -Naur orig/clutter/json/json-object.c patched/clutter/json/json-object.c
--- orig/clutter/json/json-object.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/json/json-object.c	2015-07-19 15:49:03.788501794 +0200
@@ -677,7 +677,7 @@
  *
  * Since: 0.8
  */
-G_CONST_RETURN gchar *
+const gchar *
 json_object_get_string_member (JsonObject  *object,
                                const gchar *member_name)
 {
diff -Naur orig/clutter/json/json-types.h patched/clutter/json/json-types.h
--- orig/clutter/json/json-types.h	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/json/json-types.h	2015-07-19 15:48:51.836442527 +0200
@@ -132,7 +132,7 @@
 JsonNodeType          json_node_get_node_type   (JsonNode     *node);
 GType                 json_node_get_value_type  (JsonNode     *node);
 JsonNode *            json_node_get_parent      (JsonNode     *node);
-G_CONST_RETURN gchar *json_node_type_name       (JsonNode     *node);
+const gchar *json_node_type_name       (JsonNode     *node);
 
 void                  json_node_set_object      (JsonNode     *node,
                                                  JsonObject   *object);
@@ -152,7 +152,7 @@
                                                  GValue       *value);
 void                  json_node_set_string      (JsonNode     *node,
                                                  const gchar  *value);
-G_CONST_RETURN gchar *json_node_get_string      (JsonNode     *node);
+const gchar *json_node_get_string      (JsonNode     *node);
 gchar *               json_node_dup_string      (JsonNode     *node);
 void                  json_node_set_int         (JsonNode     *node,
                                                  gint64        value);
@@ -213,7 +213,7 @@
                                                       const gchar *member_name);
 gboolean              json_object_get_boolean_member (JsonObject  *object,
                                                       const gchar *member_name);
-G_CONST_RETURN gchar *json_object_get_string_member  (JsonObject  *object,
+const gchar *json_object_get_string_member  (JsonObject  *object,
                                                       const gchar *member_name);
 gboolean              json_object_get_null_member    (JsonObject  *object,
                                                       const gchar *member_name);
@@ -260,7 +260,7 @@
                                                       guint        index_);
 gboolean              json_array_get_boolean_element (JsonArray   *array,
                                                       guint        index_);
-G_CONST_RETURN gchar *json_array_get_string_element  (JsonArray   *array,
+const gchar *json_array_get_string_element  (JsonArray   *array,
                                                       guint        index_);
 gboolean              json_array_get_null_element    (JsonArray   *array,
                                                       guint        index_);
diff -Naur orig/clutter/x11/clutter-backend-x11.c patched/clutter/x11/clutter-backend-x11.c
--- orig/clutter/x11/clutter-backend-x11.c	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/x11/clutter-backend-x11.c	2015-07-19 15:49:22.744595792 +0200
@@ -715,7 +715,7 @@
  *   returned list is owned by Clutter and should not be modified or
  *   freed
  */
-G_CONST_RETURN GSList *
+const GSList *
 clutter_x11_get_input_devices (void)
 {
   ClutterDeviceManager *manager;
diff -Naur orig/clutter/x11/clutter-x11.h patched/clutter/x11/clutter-x11.h
--- orig/clutter/x11/clutter-x11.h	2010-09-24 17:25:42.000000000 +0200
+++ patched/clutter/x11/clutter-x11.h	2015-07-19 15:49:28.904626338 +0200
@@ -128,7 +128,7 @@
 ClutterStage *clutter_x11_get_stage_from_window (Window win);
 
 #ifndef CLUTTER_DISABLE_DEPRECATED
-G_CONST_RETURN GSList* clutter_x11_get_input_devices (void) G_GNUC_DEPRECATED;
+const GSList* clutter_x11_get_input_devices (void) G_GNUC_DEPRECATED;
 #endif
 
 void     clutter_x11_enable_xinput (void);
diff -Naur orig/doc/CODING_STYLE patched/doc/CODING_STYLE
--- orig/doc/CODING_STYLE	2010-09-24 17:25:42.000000000 +0200
+++ patched/doc/CODING_STYLE	2015-07-19 15:48:32.828348271 +0200
@@ -293,7 +293,7 @@
   void                  clutter_type_set_property (ClutterType  *type,
                                                    const gchar  *value,
                                                    GError      **error);
-  G_CONST_RETURN gchar *clutter_type_get_property (ClutterType  *type);
+  const gchar *clutter_type_get_property (ClutterType  *type);
 
 Public headers should never be included directly:
 
