diff -Naur orig/catfish/CatfishWindow.py patched/catfish/CatfishWindow.py
--- orig/catfish/CatfishWindow.py	2014-09-21 23:34:50.000000000 +0200
+++ patched/catfish/CatfishWindow.py	2014-11-16 17:45:54.911423658 +0100
@@ -93,7 +93,7 @@
     """The application window."""
     __gtype_name__ = "CatfishWindow"
 
-    folder_thumbnails = os.path.expanduser('~/.thumbnails/')
+    folder_thumbnails = os.path.expanduser('~/.cache/thumbnails/')
 
     filter_timerange = (0.0, 9999999999.0)
     start_date = datetime.datetime.now()
@@ -1483,7 +1483,7 @@
 
     def get_thumbnail(self, path, mime_type=None):
         """Try to fetch a thumbnail."""
-        thumbnails_directory = os.path.expanduser('~/.thumbnails/normal')
+        thumbnails_directory = os.path.expanduser('~/.cache/thumbnails/normal')
         uri = 'file://' + path
         if helpers.check_python_version(3, 0):
             uri = uri.encode('utf-8')
