diff -Naur orig/panels/color/cinnamon-color-panel.desktop.in.in mod/panels/color/cinnamon-color-panel.desktop.in.in
--- orig/panels/color/cinnamon-color-panel.desktop.in.in	2014-10-31 13:41:59.000000000 +0100
+++ mod/panels/color/cinnamon-color-panel.desktop.in.in	2014-11-07 17:43:56.305893568 +0100
@@ -4,7 +4,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GTK;Settings;X-Cinnamon-Settings-Panel;HardwareSettings
+Categories=GTK;Settings;X-Cinnamon-Settings-Panel;HardwareSettings;
 OnlyShowIn=X-Cinnamon;
 X-Cinnamon-Settings-Panel=color
 # Translators: those are keywords for the color control-center panel
