diff -Naur orig/configure patched/configure
--- orig/configure	2011-03-31 00:13:35.000000000 +0200
+++ patched/configure	2015-07-19 16:59:58.965602078 +0200
@@ -11487,6 +11487,7 @@
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
     { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"   glib-2.0 >= 2.16
+	  gthread-2.0 >= 2.16
       gobject-2.0 >= 2.10
       gdk-2.0 >= 2.10
       clutter-1.0 >= 1.0
@@ -11496,6 +11497,7 @@
 
 \""; } >&5
   ($PKG_CONFIG --exists --print-errors "   glib-2.0 >= 2.16
+	  gthread-2.0 >= 2.16
       gobject-2.0 >= 2.10
       gdk-2.0 >= 2.10
       clutter-1.0 >= 1.0
@@ -11508,6 +11510,7 @@
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
   pkg_cv_DEPS_CFLAGS=`$PKG_CONFIG --cflags "   glib-2.0 >= 2.16
+	  gthread-2.0 >= 2.16
       gobject-2.0 >= 2.10
       gdk-2.0 >= 2.10
       clutter-1.0 >= 1.0
@@ -11527,6 +11530,7 @@
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
     { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"   glib-2.0 >= 2.16
+	  gthread-2.0 >= 2.16
       gobject-2.0 >= 2.10
       gdk-2.0 >= 2.10
       clutter-1.0 >= 1.0
@@ -11536,6 +11540,7 @@
 
 \""; } >&5
   ($PKG_CONFIG --exists --print-errors "   glib-2.0 >= 2.16
+	  gthread-2.0 >= 2.16
       gobject-2.0 >= 2.10
       gdk-2.0 >= 2.10
       clutter-1.0 >= 1.0
@@ -11548,6 +11553,7 @@
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
   pkg_cv_DEPS_LIBS=`$PKG_CONFIG --libs "   glib-2.0 >= 2.16
+	  gthread-2.0 >= 2.16
       gobject-2.0 >= 2.10
       gdk-2.0 >= 2.10
       clutter-1.0 >= 1.0
@@ -11576,6 +11582,7 @@
 fi
         if test $_pkg_short_errors_supported = yes; then
 	        DEPS_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors "   glib-2.0 >= 2.16
+	  gthread-2.0 >= 2.16
       gobject-2.0 >= 2.10
       gdk-2.0 >= 2.10
       clutter-1.0 >= 1.0
@@ -11586,6 +11593,7 @@
 " 2>&1`
         else
 	        DEPS_PKG_ERRORS=`$PKG_CONFIG --print-errors "   glib-2.0 >= 2.16
+	  gthread-2.0 >= 2.16
       gobject-2.0 >= 2.10
       gdk-2.0 >= 2.10
       clutter-1.0 >= 1.0
@@ -11599,6 +11607,7 @@
 	echo "$DEPS_PKG_ERRORS" >&5
 
 	as_fn_error $? "Package requirements (   glib-2.0 >= 2.16
+	  gthread-2.0 >= 2.16
       gobject-2.0 >= 2.10
       gdk-2.0 >= 2.10
       clutter-1.0 >= 1.0
diff -Naur orig/configure.ac patched/configure.ac
--- orig/configure.ac	2011-03-31 00:10:42.000000000 +0200
+++ patched/configure.ac	2015-07-19 16:57:35.260889484 +0200
@@ -58,6 +58,7 @@
 
 PKG_CHECK_MODULES(DEPS,
   [   glib-2.0 >= 2.16
+  	  gthread-2.0 >= 2.16
       gobject-2.0 >= 2.10
       gdk-2.0 >= 2.10
       clutter-1.0 >= 1.0
