diff -Naur orig/data/dvdstyler.desktop mod/data/dvdstyler.desktop
--- orig/data/dvdstyler.desktop	2007-07-29 11:00:10.000000000 +0200
+++ mod/data/dvdstyler.desktop	2015-07-03 23:52:49.792984441 +0200
@@ -4,8 +4,8 @@
 Encoding=UTF-8
 Name=DVD Styler
 Comment=Video DVD Production
-Icon=dvdstyler.png
+Icon=dvdstyler
 Exec=dvdstyler
 Terminal=false
-Categories=Application;AudioVideo;
+Categories=AudioVideo;
 StartupNotify=true
