diff -Nur orig/setup.py mod/setup.py
--- orig/setup.py	2012-04-30 21:26:10.000000000 +0200
+++ mod/setup.py	2017-01-09 21:56:06.695805735 +0100
@@ -427,7 +427,7 @@
         pass
 
     def run(self):
-        os.system('pybabel extract . -o po/wicd.pot --sort-output')
+        os.system('pybabel2 extract . -o po/wicd.pot --sort-output')
         os.system('xgettext -L glade data/wicd.ui -j -o po/wicd.pot')
         
 class update_translations(Command):
@@ -444,7 +444,7 @@
     def run(self):
         for pofile in glob('po/*.po'):
             lang = pofile.replace('po/', '').replace('.po', '')
-            os.system('pybabel update -o %s -i po/wicd.pot -D wicd -l %s' % (pofile, lang))
+            os.system('pybabel2 update -o %s -i po/wicd.pot -D wicd -l %s' % (pofile, lang))
 
 class compile_translations(Command):
     description = 'compile po-files to binary mo'
@@ -464,7 +464,7 @@
         for pofile in glob('po/*.po'):
             lang = pofile.replace('po/', '').replace('.po', '')
             os.makedirs('translations/' + lang + '/LC_MESSAGES/')
-            os.system('pybabel compile -D wicd -i %s -l %s -d translations/' % (pofile, lang))
+            os.system('pybabel2 compile -D wicd -i %s -l %s -d translations/' % (pofile, lang))
         
 
 class uninstall(Command):
