diff -Naur orig/src/common/utils.c patched/src/common/utils.c
--- orig/src/common/utils.c	2014-10-20 22:05:35.000000000 +0200
+++ patched/src/common/utils.c	2014-10-23 19:37:38.355183478 +0200
@@ -5215,7 +5215,7 @@
 				}
 				break;
 			case 'r':
-				strftime(subbuf, 64, "%r", lt);
+				strftime(subbuf, 64, "%X", lt);
 				len = strlen(subbuf); CHECK_SIZE();
 				strncpy2(curpos, subbuf, buflen - total_done);
 				break;
