diff -Naur orig/src/textview.c patched/src/textview.c
--- orig/src/textview.c	2015-06-23 17:34:46.000000000 +0200
+++ patched/src/textview.c	2015-07-18 23:53:16.243142942 +0200
@@ -2252,7 +2252,8 @@
 				procheader_headername_equal(header->name,
 						 "X-Newsreader")) &&
 				(strstr(header->body, "Claws Mail") != NULL ||
-				strstr(header->body, "Sylpheed-Claws") != NULL)) {
+				strstr(header->body, "Sylpheed-Claws")) &&
+				strstr(header->body, "uhu-linux") != NULL) {
 			gtk_text_buffer_get_end_iter (buffer, &iter);
 			gtk_text_buffer_insert_with_tags_by_name
 				(buffer, &iter, header->body, -1,
