diff -Naur release/src/FbTk/Menu.cc revert/src/FbTk/Menu.cc
--- release/src/FbTk/Menu.cc	2015-02-08 11:44:45.000000000 +0100
+++ revert/src/FbTk/Menu.cc	2015-03-13 15:14:53.468755230 +0100
@@ -419,17 +419,14 @@
 
 
     // calculate needed item width
-    m_item_w = 1;
-    if (m_title.visible) {
-        m_item_w = theme()->titleFont().textWidth(m_title.label);
-        m_item_w += bevel * 2;
-    }
-    m_item_w = std::max(iw, m_item_w);
-    for (i = 0; i < l; i++) {
-        iw = m_items[i]->width(theme());
-        m_item_w = std::max(iw, m_item_w);
+    unsigned int ii = 0;
+     for (i = 0; i < l; i++) {
+        ii = m_items[i]->width(theme());
+        m_item_w = (ii > m_item_w ? ii : m_item_w);
     }
 
+    if (m_item_w < 1)
+        m_item_w = 1;
 
     // calculate needed columns
     m_columns = 0;
