diff -Naur orig/apt-pkg/acquire-item.cc patched/apt-pkg/acquire-item.cc
--- orig/apt-pkg/acquire-item.cc	2015-11-27 16:06:20.000000000 +0100
+++ patched/apt-pkg/acquire-item.cc	2015-11-29 15:14:31.797448172 +0100
@@ -178,7 +178,7 @@
 
    if (_config->FindB("Acquire::AllowInsecureRepositories") == true)
    {
-      MessageInsecureRepository(false, msg, repo);
+//      MessageInsecureRepository(false, msg, repo);
       return true;
    }
 
