diff -Naur orig/apt-pkg/contrib/fileutl.h patched/apt-pkg/contrib/fileutl.h
--- orig/apt-pkg/contrib/fileutl.h	2015-11-28 22:49:07.000000000 +0100
+++ patched/apt-pkg/contrib/fileutl.h	2015-11-29 15:42:22.373732112 +0100
@@ -196,7 +196,7 @@
  *
  * @param requester is printed as functionname in error cases
  * @param file is the file to be modified
- * @param user is the (new) owner of the file, e.g. _apt
+ * @param user is the (new) owner of the file, e.g. apt
  * @param group is the (new) group owning the file, e.g. root
  * @param mode is the access mode of the file, e.g. 0644
  */
@@ -205,7 +205,7 @@
 /**
  * \brief Drop privileges
  *
- * Drop the privileges to the user _apt (or the one specified in
+ * Drop the privileges to the user apt (or the one specified in
  * APT::Sandbox::User). This does not set the supplementary group
  * ids up correctly, it only uses the default group. Also prevent
  * the process from gaining any new privileges afterwards, at least
diff -Naur orig/apt-pkg/init.cc patched/apt-pkg/init.cc
--- orig/apt-pkg/init.cc	2015-11-28 22:49:07.000000000 +0100
+++ patched/apt-pkg/init.cc	2015-11-29 15:40:26.633158186 +0100
@@ -96,7 +96,7 @@
    Cnf.CndSet("Acquire::cdrom::mount", "/media/cdrom/");
 
    // The default user we drop to in the methods
-   Cnf.CndSet("APT::Sandbox::User", "_apt");
+   Cnf.CndSet("APT::Sandbox::User", "apt");
 
    Cnf.CndSet("Acquire::IndexTargets::deb::Packages::MetaKey", "$(COMPONENT)/binary-$(ARCHITECTURE)/Packages");
    Cnf.CndSet("Acquire::IndexTargets::deb::Packages::flatMetaKey", "Packages");
diff -Naur orig/doc/sources.list.5.xml patched/doc/sources.list.5.xml
--- orig/doc/sources.list.5.xml	2015-11-28 22:49:07.000000000 +0100
+++ patched/doc/sources.list.5.xml	2015-11-29 15:40:41.281230822 +0100
@@ -283,7 +283,7 @@
 
 	  <listitem><para><option>Signed-By</option> (<option>signed-by</option>)
 		is either an absolute path to a keyring file (has to be
-		accessible and readable for the <literal>_apt</literal> user,
+		accessible and readable for the <literal>apt</literal> user,
 		so ensure everyone has read-permissions on the file) or a
 		fingerprint of a key either in the
 		<filename>trusted.gpg</filename> keyring or in one of the
