diff -Naur orig/configure.ac patched/configure.ac
--- orig/configure.ac	2014-06-25 11:50:40.000000000 +0200
+++ patched/configure.ac	2014-07-16 10:39:52.011627669 +0200
@@ -489,9 +489,9 @@
 
 AC_SUBST(GDK_PIXBUF_CSOURCE)
 
-AC_PATH_PROG(MATEDIALOG, matedialog, no)
-if test x"$MATEDIALOG" = xno; then
-  AC_MSG_ERROR([matedialog not found in your path - needed for dialogs])
+AC_PATH_PROG(ZENITY, zenity, no)
+if test x"$ZENITY" = xno; then
+  AC_MSG_ERROR([zenity not found in your path - needed for dialogs])
 fi
 
 AC_ARG_ENABLE(debug,
diff -Naur orig/src/core/util.c patched/src/core/util.c
--- orig/src/core/util.c	2014-06-25 11:42:54.000000000 +0200
+++ patched/src/core/util.c	2014-07-16 10:37:08.525351170 +0200
@@ -559,7 +559,7 @@
                                  g_slist_length (columns)*2 +
                                  g_slist_length (entries)));
 
-  argvl[i++] = "matedialog";
+  argvl[i++] = "zenity";
   argvl[i++] = type;
   argvl[i++] = "--screen";
   argvl[i++] = screen_number_text;
