diff -Naur orig/src/Makefile.am patched/src/Makefile.am
--- orig/src/Makefile.am	2014-07-29 18:42:01.000000000 +0200
+++ patched/src/Makefile.am	2014-11-22 18:52:08.151414037 +0100
@@ -201,7 +201,8 @@
 	$(toolbar_LIB)			\
 	$(screensaver_LIB)		\
 	$(jpeg_LIB)			\
-	$(INTROSPECTION_LIBS)
+	$(INTROSPECTION_LIBS) \
+	-lutil
 
 BUILT_SOURCES = 			\
 	eom-enum-types.c		\
diff -Naur orig/src/Makefile.in patched/src/Makefile.in
--- orig/src/Makefile.in	2014-10-24 10:42:52.000000000 +0200
+++ patched/src/Makefile.in	2014-11-22 18:53:15.095745996 +0100
@@ -629,7 +629,8 @@
 	$(toolbar_LIB)			\
 	$(screensaver_LIB)		\
 	$(jpeg_LIB)			\
-	$(INTROSPECTION_LIBS)
+	$(INTROSPECTION_LIBS) \
+	-lutil
 
 BUILT_SOURCES = eom-enum-types.c eom-enum-types.h $(MARSHAL_OUTPUT) \
 	$(am__append_11)
