diff -Naur orig/src/Makefile.am patched/src/Makefile.am
--- orig/src/Makefile.am	2013-03-24 15:56:56.000000000 +0100
+++ patched/src/Makefile.am	2014-07-17 00:17:54.670088323 +0200
@@ -194,7 +194,8 @@
 	$(LIBJPEG)			\
 	$(toolbar_LIB)			\
 	$(screensaver_LIB)		\
-	$(jpeg_LIB)
+	$(jpeg_LIB)			\
+	-lutil
 
 eom_LDFLAGS = -export-dynamic -no-undefined -export-symbols-regex "^[[^_]].*"
 
diff -Naur orig/src/Makefile.in patched/src/Makefile.in
--- orig/src/Makefile.in	2014-03-01 13:18:29.000000000 +0100
+++ patched/src/Makefile.in	2014-07-17 00:18:31.846485157 +0200
@@ -606,7 +606,8 @@
 	$(LIBJPEG)			\
 	$(toolbar_LIB)			\
 	$(screensaver_LIB)		\
-	$(jpeg_LIB)
+	$(jpeg_LIB)			\
+	-lutil
 
 eom_LDFLAGS = -export-dynamic -no-undefined -export-symbols-regex "^[[^_]].*"
 BUILT_SOURCES = eom-enum-types.c eom-enum-types.h $(MARSHAL_OUTPUT) \
