diff -Naur orig/data/lightdm-gtk-greeter.conf mod/data/lightdm-gtk-greeter.conf
--- orig/data/lightdm-gtk-greeter.conf	2014-08-18 04:35:10.000000000 +0200
+++ mod/data/lightdm-gtk-greeter.conf	2014-09-15 15:23:58.261712496 +0200
@@ -27,7 +27,7 @@
 # laptop = true|false, false by default. Marks monitor as laptop display.
 # 
 [greeter]
-#background=
+background=/usr/share/wallpapers/UHU/uhu-3-bg.png
 #user-background=
 #theme-name=
 #icon-theme-name=
@@ -37,9 +37,9 @@
 #xft-hintstyle=
 #xft-rgba=
 #show-indicators=
-#show-clock=
+show-clock=true
 #clock-format=
 #keyboard=
 #reader=
-#position=
+position= 60 80
 #screensaver-timeout=
