diff -Naur orig/data/lightdm-gtk-greeter.conf patched/data/lightdm-gtk-greeter.conf
--- orig/data/lightdm-gtk-greeter.conf	2015-02-16 01:34:19.000000000 +0100
+++ patched/data/lightdm-gtk-greeter.conf	2015-02-28 18:20:13.885434498 +0100
@@ -45,7 +45,7 @@
 #  laptop = false|true ("false" by default) Marks monitor as laptop display
 # 
 [greeter]
-#background=
+background=/usr/share/wallpapers/UHU/uhu-3-bg.png
 #theme-name=
 #icon-theme-name=
 #font-name=
@@ -54,9 +54,9 @@
 #xft-hintstyle=
 #xft-rgba=
 #show-indicators=
-#show-clock=
+show-clock=true
 #clock-format=
 #keyboard=
 #reader=
-#position=
+position= 60 80
 #screensaver-timeout=
