diff -Naur orig/files/usr/share/cinnamon-session/sessions/cinnamon2d.session mod/files/usr/share/cinnamon-session/sessions/cinnamon2d.session
--- orig/files/usr/share/cinnamon-session/sessions/cinnamon2d.session	2014-10-31 23:21:43.000000000 +0100
+++ mod/files/usr/share/cinnamon-session/sessions/cinnamon2d.session	2014-11-08 18:23:48.109255609 +0100
@@ -1,6 +1,6 @@
 [Cinnamon Session]
 Name=Cinnamon (Software Rendering)
-RequiredComponents=cinnamon2d;cinnamon-settings-daemon;cinnamon-screensaver;nemo-autostart;nm-applet;
+RequiredComponents=cinnamon2d;cinnamon-settings-daemon;cinnamon-screensaver;nemo-autostart;nm-applet;polkit-cinnamon-authentication-agent-1;
 DesktopName=X-Cinnamon
 
 
diff -Naur orig/files/usr/share/cinnamon-session/sessions/cinnamon.session mod/files/usr/share/cinnamon-session/sessions/cinnamon.session
--- orig/files/usr/share/cinnamon-session/sessions/cinnamon.session	2014-10-31 23:21:43.000000000 +0100
+++ mod/files/usr/share/cinnamon-session/sessions/cinnamon.session	2014-11-08 18:23:36.541256069 +0100
@@ -1,6 +1,6 @@
 [Cinnamon Session]
 Name=Cinnamon
-RequiredComponents=cinnamon;cinnamon-settings-daemon;cinnamon-screensaver;nemo-autostart;nm-applet;
+RequiredComponents=cinnamon;cinnamon-settings-daemon;cinnamon-screensaver;nemo-autostart;nm-applet;polkit-cinnamon-authentication-agent-1;
 DesktopName=X-Cinnamon
 
 
