diff -Naur orig/src-qt5/OS-detect.pri patched/src-qt5/OS-detect.pri
--- orig/src-qt5/OS-detect.pri	2016-05-03 18:03:16.000000000 +0200
+++ patched/src-qt5/OS-detect.pri	2016-06-18 02:08:02.711174482 +0200
@@ -80,7 +80,7 @@
   # If the detailed install variables are not set - create them from the general vars
   isEmpty(L_BINDIR){ L_BINDIR = $${PREFIX}/bin }
   isEmpty(L_LIBDIR){ L_LIBDIR = $${PREFIX}/lib }
-  isEmpty(L_ETCDIR){ L_ETCDIR = $${PREFIX}/etc }
+  isEmpty(L_ETCDIR){ L_ETCDIR = /etc }
   isEmpty(L_SHAREDIR){ L_SHAREDIR = $${PREFIX}/share }
   isEmpty(L_INCLUDEDIR){ L_INCLUDEDIR = $${PREFIX}/include }
   isEmpty(L_SESSDIR){ L_SESSDIR = $${L_SHAREDIR}/xsessions }
