diff -Naur orig/configure patched/configure
--- orig/configure	2015-09-17 16:51:51.000000000 +0200
+++ patched/configure	2015-09-17 23:12:06.548648005 +0200
@@ -12932,8 +12932,8 @@
 # ***************************
 
 GLIB_REQUIRED=2.36.0
-POLKIT_AGENT_REQUIRED=0.97
-POLKIT_GOBJECT_REQUIRED=0.97
+POLKIT_AGENT_REQUIRED=0.102
+POLKIT_GOBJECT_REQUIRED=0.102
 APPINDICATOR_REQUIRED=0.0.13
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking which gtk+ version to compile against" >&5
diff -Naur orig/configure.ac patched/configure.ac
--- orig/configure.ac	2015-09-17 16:51:16.000000000 +0200
+++ patched/configure.ac	2015-09-17 23:12:21.572722505 +0200
@@ -121,8 +121,8 @@
 # ***************************
 
 GLIB_REQUIRED=2.36.0
-POLKIT_AGENT_REQUIRED=0.97
-POLKIT_GOBJECT_REQUIRED=0.97
+POLKIT_AGENT_REQUIRED=0.102
+POLKIT_GOBJECT_REQUIRED=0.102
 APPINDICATOR_REQUIRED=0.0.13
 
 AC_MSG_CHECKING([which gtk+ version to compile against])
diff -Naur orig/polkitgtkmate/example.c patched/polkitgtkmate/example.c
--- orig/polkitgtkmate/example.c	2015-09-17 16:51:16.000000000 +0200
+++ patched/polkitgtkmate/example.c	2015-09-17 23:13:44.881135609 +0200
@@ -147,7 +147,7 @@
 
 
 	system_bus_name_subject = polkit_system_bus_name_new(g_dbus_connection_get_unique_name(bus));
-	unix_process_subject = polkit_unix_process_new(getpid());
+	unix_process_subject = polkit_unix_process_new_for_owner(getpid(), 0, -1);
 
 	error = NULL;
 	authority = polkit_authority_get_sync(NULL /* GCancellable* */, &error);
diff -Naur orig/polkitgtkmate/polkitlockbutton.c patched/polkitgtkmate/polkitlockbutton.c
--- orig/polkitgtkmate/polkitlockbutton.c	2015-09-17 16:51:16.000000000 +0200
+++ patched/polkitgtkmate/polkitlockbutton.c	2015-09-17 23:14:19.293306250 +0200
@@ -418,7 +418,7 @@
 
   if (button->priv->subject == NULL)
     {
-      button->priv->subject = polkit_unix_process_new (getpid ());
+      button->priv->subject = polkit_unix_process_new_for_owner (getpid(), 0, -1);
     }
 
   /* synchronously check on construction - TODO: we could implement GAsyncInitable
