diff -Naur orig/lib/menu.in mod/lib/menu.in
--- orig/lib/menu.in	2015-07-03 03:48:23.000000000 +0200
+++ mod/lib/menu.in	2015-07-07 17:56:51.933451848 +0200
@@ -4,14 +4,11 @@
 # since modifications to this file will be discarded when you
 # (re)install icewm.
 #
-prog xterm xterm xterm
-prog rxvt xterm rxvt -bg black -cr green -fg white -C -fn 9x15 -sl 500
-prog fte fte fte
-prog NEdit nedit nedit
-prog Mozilla mozilla mozilla
-prog XChat xchat xchat
-prog Gimp gimp gimp
+prog xterm /usr/share/icons/gnome/16x16/apps/terminal.png xterm
+prog "Web böngésző" /usr/share/icons/gnome/16x16/apps/web-browser.png xdg-open about:blank
+
 separator
 menuprog "Desktop Apps" folder icewm-menu-fdo
-menufile Programs folder programs
-menufile Tool_bar folder toolbar
+prog "Menü újraépítése" /usr/share/icons/gnome/16x16/actions/edit-find-replace.png /usr/share/icewm/startup
+separator
+menufile Alkalmazások folder programs.autogen
