diff -Naur orig/CMakeLists.txt mod/CMakeLists.txt
--- orig/CMakeLists.txt	2015-07-11 14:37:31.000000000 +0200
+++ mod/CMakeLists.txt	2015-07-11 17:36:52.493826110 +0200
@@ -1,4 +1,7 @@
 cmake_minimum_required(VERSION 2.6)
+IF (POLICY CMP0053) # in CMake 3.1.0+
+  CMAKE_POLICY (SET CMP0053 OLD) # keep old-style @VAR@ expansion
+ENDIF (POLICY CMP0053)
 PROJECT(ICEWM CXX C)
 
 set(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR})
